
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Versioned history of the HTAN data model.
">
      
      
      
      
        <link rel="prev" href="..">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.5.49">
    
    
      
        <title>Dictionary - HTAN Data Model Dictionary</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.6f8fc17f.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      
  
  
    
    
  
    
    
  
  
  <style>:root{--md-admonition-icon--note:url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M1 7.775V2.75C1 1.784 1.784 1 2.75 1h5.025c.464 0 .91.184 1.238.513l6.25 6.25a1.75 1.75 0 0 1 0 2.474l-5.026 5.026a1.75 1.75 0 0 1-2.474 0l-6.25-6.25A1.75 1.75 0 0 1 1 7.775m1.5 0c0 .066.026.13.073.177l6.25 6.25a.25.25 0 0 0 .354 0l5.025-5.025a.25.25 0 0 0 0-.354l-6.25-6.25a.25.25 0 0 0-.177-.073H2.75a.25.25 0 0 0-.25.25ZM6 5a1 1 0 1 1 0 2 1 1 0 0 1 0-2"/></svg>');--md-admonition-icon--info:url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8m8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13M6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75M8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2"/></svg>');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="HTAN Data Model Dictionary" class="md-header__button md-logo" aria-label="HTAN Data Model Dictionary" data-md-component="logo">
      
  <img src="../assets/htan-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            HTAN Data Model Dictionary
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Dictionary
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6m0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4M7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="deep-purple" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/ncihtan/data-models" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    data-models
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="./" class="md-tabs__link">
          
  
  Dictionary

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="HTAN Data Model Dictionary" class="md-nav__button md-logo" aria-label="HTAN Data Model Dictionary" data-md-component="logo">
      
  <img src="../assets/htan-logo.png" alt="logo">

    </a>
    HTAN Data Model Dictionary
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/ncihtan/data-models" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    data-models
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
      
        
        
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Dictionary
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Dictionary
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Dictionary
  </span>
  

      </a>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Dictionary</h1>

<table>
<thead>
<tr>
<th style="text-align: left;">Attribute</th>
<th style="text-align: left;">Description</th>
<th style="text-align: left;">DependsOn</th>
<th style="text-align: left;">Required</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;">Assay</td>
<td style="text-align: left;">A planned process with the objective to produce information about the material entity that is the evaluant, by physically examining it or its proxies.[OBI_0000070]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Device</td>
<td style="text-align: left;">A thing made or adapted for a particular purpose, especially a piece of mechanical or electronic equipment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sequencing</td>
<td style="text-align: left;">Module for next generation sequencing assays</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Component</td>
<td style="text-align: left;">Category of metadata (e.g. Diagnosis, Biospecimen, scRNA-seq Level 1, etc.); provide the same one for all items/rows.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Patient</td>
<td style="text-align: left;">HTAN patient</td>
<td style="text-align: left;">Component, HTAN Participant ID</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">File</td>
<td style="text-align: left;">A type of Information Content Entity specific to OS</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Filename</td>
<td style="text-align: left;">Name of a file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">File Format</td>
<td style="text-align: left;">Format of a file (e.g. txt, csv, fastq, bam, etc.)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS Sequencing Template</td>
<td style="text-align: left;">CDS compatible template file, includes attributes for Genomic Reference, Library Layout, Data Type, Sequencing Platform, Library Selection Method</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, CDS library_id, CDS library_strategy, CDS library_source, CDS library_selection, CDS library_layout, CDS platform, CDS instrument_model, CDS design_description, CDS reference_genome_assembly, CDS custom_assembly_fasta_file_for_alignment, CDS bases, CDS number_of_reads, CDS coverage, CDS avg_read_length, CDS sequence_alignment_software</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS library_id</td>
<td style="text-align: left;">Short unique identifier for the sequencing library.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS library_strategy</td>
<td style="text-align: left;">Library strategy</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS library_source</td>
<td style="text-align: left;">The Library Source specifies the type of source material that is being sequenced</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS library_selection</td>
<td style="text-align: left;">Library Selection Method</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS library_layout</td>
<td style="text-align: left;">Paired-end or Single</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS platform</td>
<td style="text-align: left;">Sequencing Platform used for Sequencing</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS instrument_model</td>
<td style="text-align: left;">Instrument model used for sequencing</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CDS design_description</td>
<td style="text-align: left;">Free-form description of the methods used to create the sequencing library; a brief 'materials and methods' section.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS reference_genome_assembly</td>
<td style="text-align: left;">This is only if you are submitting a bam file aligned against a NCBI assembly.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS custom_assembly_fasta_file_for_alignment</td>
<td style="text-align: left;">Please provide the name of the custom assembly fasta file used during alignment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS bases</td>
<td style="text-align: left;">Count of unique basecalls present in the data. Please count each base only once if using secondary alignments.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS number_of_reads</td>
<td style="text-align: left;">Count of the number of reads in the data. Please count each read only once if using secondary alignments.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS coverage</td>
<td style="text-align: left;">Depth of coverage on assembly used. Found by (Unique Aligned Basecalls)/(Reference Length)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS avg_read_length</td>
<td style="text-align: left;">Found by (Bases)/(Reads)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDS sequence_alignment_software</td>
<td style="text-align: left;">The name of the software program used to align nucleotide sequencing data.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Checksum</td>
<td style="text-align: left;">MD5 checksum of the BAM file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Data File ID</td>
<td style="text-align: left;">Self-identifier for this data file - HTAN ID of this file HTAN ID SOP (eg HTANx_yyy_zzz)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Participant ID</td>
<td style="text-align: left;">HTAN ID associated with a patient based on HTAN ID SOP (eg HTANx_yyy )</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Biospecimen ID</td>
<td style="text-align: left;">HTAN ID associated with a biosample based on HTAN ID SOP (eg HTANx_yyy_zzz)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Parent ID</td>
<td style="text-align: left;">HTAN ID of parent from which the biospecimen was obtained. Parent could be another biospecimen or a research participant.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Parent Biospecimen ID</td>
<td style="text-align: left;">HTAN Biospecimen Identifier (eg HTANx_yyy_zzz) indicating the biospecimen(s) from which these files were derived; multiple parent biospecimen should be comma-separated</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Parent Data File ID</td>
<td style="text-align: left;">HTAN Data File Identifier indicating the file(s) from which these files were derived</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Clinical Data Tier 2</td>
<td style="text-align: left;">Tier 2 Cancer Data</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index, Sentinel Lymph Node Count, Sentinel Node Positive Assessment Count, Tumor Extranodal Extension Indicator, Satellite Metastasis Present Indicator, Other Biopsy Resection Site, Extent of Tumor Resection, Prior Sites of Radiation, Immunosuppression, Concomitant Medication Received Type, Family Member Vital Status Indicator, COVID19 Occurrence Indicator, COVID19 Current Status, COVID19 Positive Lab Test Indicator, COVID19 Antibody Testing, COVID19 Complications Severity, COVID19 Cancer Treatment Followup, Ecig vape use, Ecig vape 30 day use num, Ecig vape times per day, Type of smoke exposure cumulative years, Chewing tobacco daily use count, Second hand smoke exposure years, Known Genetic Predisposition Mutation, Hereditary Cancer Predisposition Syndrome, Cancer Associated Gene Mutations, Mutational Signatures, Mismatch Repair System Status, Lab Tests for MMR Status, Mode of Cancer Detection, Education Level, Country of Birth, Medically Underserved Area, Rural vs Urban, Cancer Incidence, Cancer Incidence Location</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">SRRS Clinical Data Tier 2</td>
<td style="text-align: left;">Cancer related clinical data specific to SRRS</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index, Education Level, Country of Birth, Medically Underserved Area, Rural vs Urban, Cancer Incidence, Cancer Incidence Location, Ethnicity, Gender, Race, Vital Status, Age at Diagnosis, Days to Last Follow up, Days to Last Known Disease Status, Days to Recurrence, Last Known Disease Status, Morphology, Primary Diagnosis, Progression or Recurrence, Site of Resection or Biopsy, Tissue or Organ of Origin, NCI Atlas Cancer Site, Tumor Grade, Pack Years Smoked, Years Smoked, Days to Follow Up, Gene Symbol, Molecular Analysis Method, Test Result, Treatment Type, Tumor Largest Dimension Diameter</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Cancer Tier 3</td>
<td style="text-align: left;">Lung cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index, Lung Cancer Detection Method Type, Lung Cancer Participant Procedure History, Lung Adjacent Histology Type, Lung Tumor Location Anatomic Site, Lung Tumor Lobe Bronchial Location, Current Lung Cancer Symptoms, Lung Topography, Lung Cancer Harboring Genomic Aberrations</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Cancer Tier 3</td>
<td style="text-align: left;">Colorectal cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Colorectal Cancer Detection Method Type, History of Prior Colon Polyps, Family Colon Cancer History Indicator, Family Medical History Colorectal Polyp Diagnosis, Immediate Family History Endometrial Cancer, Immediate Family History Ovarian Cancer, Patient Inflammatory Bowel Disease Personal Medica History, Patient Colonoscopy Performed Indicator, Colorectal Cancer Tumor Border Configuration, MLH1 Promoter Methylation Status, Colorectal Cancer KRAS Indicator, Colon Polyp Occurence Indicator, Family History Colorectal Polyp, Colorectal Polyp New Indicator, Colorectal Polyp Shape, Size of Polyp Removed, Colorectal Polyp Count, Colorectal Polyp Type, Colorectal Polyp Adenoma Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Cancer Tier 3</td>
<td style="text-align: left;">Breast cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Breast Carcinoma Detection Method Type, Breast Carcinoma Histology Category, Invasive Lobular Breast Carcinoma Histologic Category, Invasive Ductal Breast Carcinoma Histologic Category, Breast Biopsy Procedure Finding Type, Breast Quadrant Site, Breast Cancer Assessment Tests, Breast Cancer Genomic Test Performed, Mammaprint Risk Group, Oncotype Risk Group, Breast Carcinoma Estrogen Receptor Status, Breast Carcinoma Progesteroner Receptor Status, Breast Cancer Allred Estrogen Receptor Score, Prior Invasive Breast Disease, Breast Carcinoma ER Status Percentage Value, Breast Carcinoma PR Status Percentage Value, HER2 Breast Carcinoma Copy Number Total, Breast Carcinoma Centromere 17 Copy Number, Breast Carcinoma HER2 Centromere17 Copynumber Total, Breast Carcinoma HER2 Chromosome17 Ratio, Breast Carcinoma Surgical Procedure Name, Breast Carcinoma HER2 Ratio Diagnosis, Breast Carcinoma HER2 Status, Hormone Therapy Breast Cancer Prevention Indicator, Breast Carcinoma ER Staining Intensity, Breast Carcinoma PR Staining Intensity, Oncotype Score, Breast Imaging Performed Type, Multifocal Breast Carcinoma Present Indicator, Multicentric Breast Carcinoma Present Indicator, BIRADS Mammography Breast Density Category</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Neuroblastoma and Glioma Tier 3</td>
<td style="text-align: left;">Brain cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,CNS Tumor Primary Anatomic Site, Glioma Specific Metastasis Sites, Glioma Specific Radiation Field, Supra Tentorial Ependymoma Molecular Subgroup, Infra Tentorial Ependymoma Molecular Subgroup, Neuroblastoma MYCN Gene Amplification Status</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Acute Lymphoblastic Leukemia Tier 3</td>
<td style="text-align: left;">Acute Lymphoblastic Leukemia attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Specimen Blast Count Percentage Value, NCI ALL Risk Group, MRD ALL Diagnostic Sensitivity, CNS Leukemia Status</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Cancer Tier 3</td>
<td style="text-align: left;">Ovarian cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Ovarian Cancer Histologic Subtype, Ovarian Cancer Surgical Outcome, Ovarian Cancer Platinum Status</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prostate Cancer Tier 3</td>
<td style="text-align: left;">Prostate cancer specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Location Extent Extraprostatic Extension, Location Nature Positive Margins, Seminal Vesicle Invasion, Prostate Carcinoma Histologic Type, Prostate Cancer Local Extent, Additonal Findings Uninvolved Prostate, Prostate Cancer Cytologic Morphologic Subtypes</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sarcoma Tier 3</td>
<td style="text-align: left;">Sarcoma specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Sarcoma Subtype, Sarcoma Diagnosis Classification Category, Sarcoma Tumor Extension Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pancreatic Cancer Tier 3</td>
<td style="text-align: left;">Pancreatic cancer specific attributes in Clinical Tier Data 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index,Pancreas Precancer Histopathologic Grade, Pancreatic IPMN Pathology Epithelial Subtype, Pancreatic Duct Final Pathology Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Melanoma Tier 3</td>
<td style="text-align: left;">Melanoma specific attributes in Clinical Data Tier 3</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index, Cutaneous Melanoma Tumor Infiltrating Lymphocytes, Cutaneous Melanoma Tumor Regression Range, Melanoma Specimen Clark Level Value, Cutaneous Melanoma Surgical Margins, Melanoma Lesion Size, History of Atypical Nevi, Fitzpatrick Skin Tone, History of Chronic UV Exposure, History of Blistering Sunburn, History of Tanning Bed Use, Immediate Family History Melanoma, Melanoma Biopsy Resection Sites, Cutaneous Melanoma Ulceration, Cutaneous Melanoma Additional Findings</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Demographics</td>
<td style="text-align: left;">Demographic attributes</td>
<td style="text-align: left;">Component, HTAN Participant ID, Ethnicity, Gender, Race, Vital Status, Days to Birth, Country of Residence, Age Is Obfuscated, Year Of Birth, Occupation Duration Years, Premature At Birth, Weeks Gestation at Birth</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Family History</td>
<td style="text-align: left;">Family cancer history</td>
<td style="text-align: left;">Component, HTAN Participant ID, Relative with Cancer History</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Exposure</td>
<td style="text-align: left;">Exposure to carcinogens</td>
<td style="text-align: left;">Component, HTAN Participant ID, Start Days from Index, Smoking Exposure, Alcohol Exposure, Asbestos Exposure, Coal Dust Exposure, Environmental Tobacco Smoke Exposure, Radon Exposure, Respirable Crystalline Silica Exposure</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Follow Up</td>
<td style="text-align: left;">Follow up clinical visits</td>
<td style="text-align: left;">Component, HTAN Participant ID, Days to Follow Up, Adverse Event, Progression or Recurrence, Barretts Esophagus Goblet Cells Present, BMI, Cause of Response, Comorbidity, Comorbidity Method of Diagnosis, Days to Adverse Event, Days to Comorbidity, Diabetes Treatment Type, Disease Response, DLCO Ref Predictive Percent, ECOG Performance Status, FEV1 FVC Post Bronch Percent, FEV 1 FVC Pre Bronch Percent, FEV1 Ref Post Bronch Percent, FEV1 Ref Pre Bronch Percent, Height, Hepatitis Sustained Virological Response, HPV Positive Type, Karnofsky Performance Status, Menopause Status, Pancreatitis Onset Year,  Reflux Treatment Type, Risk Factor, Risk Factor Treatment, Viral Hepatitis Serologies, Weight, Adverse Event Grade, AIDS Risk Factors, Body Surface Area, CD4 Count, CDC HIV Risk Factors, Days to Imaging, Evidence of Recurrence Type, HAART Treatment Indicator, HIV Viral Load, Hormonal Contraceptive Use, Hysterectomy Margins Involved, Hysterectomy Type, Imaging Result, Imaging Type, Immunosuppressive Treatment Type, Nadir CD4 Count, Pregnancy Outcome, Recist Targeted Regions Number, Recist Targeted Regions Sum, Scan Tracer Used</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Therapy</td>
<td style="text-align: left;">Clinical therapy or treatment</td>
<td style="text-align: left;">Component, HTAN Participant ID, Treatment or Therapy, Treatment Type, Treatment Effect, Treatment Outcome, Days to Treatment End, Treatment Anatomic Site, Days to Treatment Start, Initial Disease Status, Regimen or Line of Therapy, Therapeutic Agents, Treatment Intent Type, Chemo Concurrent to Radiation, Number of Cycles, Reason Treatment Ended, Treatment Arm, Treatment Dose, Treatment Dose Units, Treatment Effect Indicator, Treatment Frequency</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Diagnosis</td>
<td style="text-align: left;">Disease diagnosis</td>
<td style="text-align: left;">Component, HTAN Participant ID, Age at Diagnosis, Year of Diagnosis, Primary Diagnosis, Precancerous Condition Type, Site of Resection or Biopsy, Tissue or Organ of Origin, Morphology, Tumor Grade, Progression or Recurrence, Last Known Disease Status, Days to Last Follow up, Days to Last Known Disease Status,  Method of Diagnosis, Prior Malignancy, Prior Treatment, Metastasis at Diagnosis, Metastasis at Diagnosis Site, First Symptom Prior to Diagnosis, Days to Diagnosis, Percent Tumor Invasion, Residual Disease, Synchronous Malignancy, Tumor Confined to Organ of Origin, Tumor Focality, Tumor Largest Dimension Diameter, Gross Tumor Weight, Breslow Thickness, Vascular Invasion Present, Vascular Invasion Type, Anaplasia Present, Anaplasia Present Type, Laterality, Perineural Invasion Present, Lymphatic Invasion Present, Lymph Nodes Positive, Lymph Nodes Tested, Peritoneal Fluid Cytological Status, Classification of Tumor, Best Overall Response, Mitotic Count, AJCC Clinical M, AJCC Clinical N, AJCC Clinical Stage, AJCC Clinical T, AJCC Pathologic M, AJCC Pathologic N, AJCC Pathologic Stage, AJCC Pathologic T, AJCC Staging System Edition, Cog Neuroblastoma Risk Group, Cog Rhabdomyosarcoma Risk Group, Gleason Grade Group, Gleason Grade Tertiary, Gleason Patterns Percent, Greatest Tumor Dimension, IGCCCG Stage, INPC Grade, INPC Histologic Group, INRG Stage, INSS Stage, International Prognostic Index, IRS Group, IRS Stage, ISS Stage, Lymph Node Involved Site, Margin Distance, Margins Involved Site, Medulloblastoma Molecular Classification, Micropapillary Features, Mitosis Karyorrhexis Index, Non Nodal Regional Disease, Non Nodal Tumor Deposits, Ovarian Specimen Status, Ovarian Surface Involvement, Pregnant at Diagnosis, Primary Gleason Grade, Secondary Gleason Grade, Supratentorial Localization, Tumor Depth, WHO CNS Grade, WHO NTE Grade</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Molecular Test</td>
<td style="text-align: left;">Clinical molecular test data</td>
<td style="text-align: left;">Component, HTAN Participant ID, Timepoint Label, Start Days from Index, Stop Days from Index, Gene Symbol, Molecular Analysis Method, Test Result, AA Change, Antigen, Clinical Biospecimen Type, Blood Test Normal Range Upper, Blood Test Normal Range Lower, Cell Count, Chromosome, Clonality, Copy Number, Cytoband, Exon, Histone Family, Histone Variant, Intron, Laboratory Test, Loci Abnormal Count, Loci Count, Locus, Mismatch Repair Mutation, Molecular Consequence, Pathogenicity, Ploidy, Second Exon, Second Gene Symbol, Specialized Molecular Test, Test Analyte Type, Test Units, Test Value, Transcript, Variant Origin, Variant Type, Zygosity</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biospecimen</td>
<td style="text-align: left;">HTAN biological entity; this can be tissue, blood, analyte and subsamples of those</td>
<td style="text-align: left;">Component, HTAN Biospecimen ID, Source HTAN Biospecimen ID, HTAN Parent ID, Timepoint Label, Collection Days from Index, Adjacent Biospecimen IDs, Biospecimen Type, Acquisition Method Type, Fixative Type, Storage Method, Processing Days from Index, Protocol Link,  Site Data Source, Collection Media, Mounting Medium, Processing Location, Histology Assessment By, Histology Assessment Medium, Preinvasive Morphology, Tumor Infiltrating Lymphocytes, Degree of Dysplasia, Dysplasia Fraction, Number Proliferating Cells, Percent Eosinophil Infiltration, Percent Granulocyte Infiltration, Percent Inflam Infiltration, Percent Lymphocyte Infiltration, Percent Monocyte Infiltration, Percent Necrosis, Percent Neutrophil Infiltration, Percent Normal Cells, Percent Stromal Cells, Percent Tumor Cells, Percent Tumor Nuclei, Fiducial Marker, Slicing Method,  Lysis Buffer, Method of Nucleic Acid Isolation</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">SRRS Biospecimen</td>
<td style="text-align: left;">SRRS-specific HTAN biological entity; this can be tissue, blood, analyte and subsamples of those, however it can be described via fewer attributes than a standard HTAN specimen</td>
<td style="text-align: left;">Component, HTAN Biospecimen ID, Source HTAN Biospecimen ID, HTAN Parent ID, Adjacent Biospecimen IDs, Biospecimen Type, Timepoint Label, Collection Days from Index, Acquisition Method Type, Ischemic Time, Ischemic Temperature, Collection Media, Topography Code, Additional Topography, Fixative Type, Storage Method, Preinvasive Morphology, Histologic Morphology Code, Preservation Method, Processing Days from Index, Protocol Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Source HTAN Biospecimen ID</td>
<td style="text-align: left;">This is the HTAN ID that may have been assigned to the biospecimen at the site of biospecimen origin (e.g. BU).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Assay</td>
<td style="text-align: left;">Metadata applying to any assay without standard descriptors. Can be used as a placeholder for minimal amount of metadata until the assay descriptors are standardized</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Assay Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">ExSeq Minimal</td>
<td style="text-align: left;">Minimal metadata for the ExSeq assay</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Assay Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Assay Type</td>
<td style="text-align: left;">The type and level of assay this metadata applies to (e.g. RPPA, NanoString DSP, etc.)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scRNA-seq Level 1</td>
<td style="text-align: left;">Single-cell RNA-seq [EFO_0008913]</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, Cryopreserved Cells in Sample, Single Cell Isolation Method, Dissociation Method, Library Construction Method, Read Indicator, Read1, Read2, End Bias, Reverse Transcription Primer, Spike In, Sequencing Platform, Total Number of Input Cells, Input Cells and Nuclei, Library Preparation Days from Index, Single Cell Dissociation Days from Index, Sequencing Library Construction Days from Index, Nucleic Acid Capture Days from Index, Protocol Link, Technical Replicate Group</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scRNA-seq Level 2</td>
<td style="text-align: left;">Alignment workflows downstream of scRNA-seq Level 1</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, scRNAseq Workflow Type, Workflow Version, scRNAseq Workflow Parameters Description, Workflow Link, Genomic Reference, Genomic Reference URL, Genome Annotation URL, Checksum, Whitelist Cell Barcode File Link, Cell Barcode Tag, UMI Tag, Applied Hard Trimming</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scRNA-seq Level 3</td>
<td style="text-align: left;">Gene and Isoform expression files</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Data Category, Matrix Type, Linked Matrices, Cell Median Number Reads, Cell Median Number Genes, Cell Total, scRNAseq Workflow Type, scRNAseq Workflow Parameters Description, Workflow Link, Workflow Version</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scRNA-seq Level 4</td>
<td style="text-align: left;">Data represents the relationships between cells derived from Level 3 expression data and shown as tSNE or UMAP coordinates per cell, plus all other cell-specific meta information (e.g., cell type)</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, scRNAseq Workflow Type, scRNAseq Workflow Parameters Description, Workflow Version, Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Level 1</td>
<td style="text-align: left;">Raw sequencing files for the Slide-seq assay.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, Read Indicator, Spatial Read1, Spatial Read2, End Bias, Reverse Transcription Primer, Spatial Barcode Offset, Spatial Barcode and UMI, Spike In, Sequencing Platform, Technical Replicate Group, Protocol Link, Spatial Library Construction Method, Library Preparation Days from Index, Sequencing Library Construction Days from Index, Nucleic Acid Capture Days from Index</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Level 2</td>
<td style="text-align: left;">Aligned sequencing files and QC for the Slide-seq assay.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Slide-seq Workflow Type, Workflow Version, Slide-seq Workflow Parameter Description, Workflow Link, Genomic Reference, Genomic Reference URL, Genome Annotation URL, Checksum, Spatial Barcode Tag, Matched Spatial Barcode Tag, UMI Tag, Applied Hard Trimming</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Level 3</td>
<td style="text-align: left;">Gene matrices with features and barcodes for Slide-seq as well as spatial information (bead location files).</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Run ID, Sequencing Batch ID, Data Category, Matrix Type, Slide-seq Workflow Type, Workflow Version, Slide-seq Workflow Parameter Description, Workflow Link, Beads Total, Median UMI Counts per Spot, Median Number Genes per Spatial Spot, Slide-seq Bead File Type, Slide-seq Fragment Size</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Fragment Size</td>
<td style="text-align: left;">Average cDNA length associated with the experiemtn. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Matched Spatial Barcode Tag</td>
<td style="text-align: left;">SAM tag for matched spot barcode field; please provide a valid spot barcode tag (e.g. CB:Z) (Slide-seq specific)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Beads Total</td>
<td style="text-align: left;">Number of sequenced beads. Applies to raw counts matrix only. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze the Slide-seq data set. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Workflow Parameter Description</td>
<td style="text-align: left;">Parameters used to run the Slide-seq workflow. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Slide-seq Bead File Type</td>
<td style="text-align: left;">The type of Level 3 file submitted as part of the Slide-seq workflow.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Bulk RNA-seq Level 1</td>
<td style="text-align: left;">Bulk RNA-seq [EFO_0003738]</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Library Layout, Read Indicator, Nucleic Acid Source, Micro-region Seq Platform, ROI Tag, Sequencing Platform, Sequencing Batch ID, Read Length, Library Selection Method, Library Preparation Kit Name, Library Preparation Kit Vendor, Library Preparation Kit Version, Library Preparation Days from Index, Spike In, Adapter Name, Adapter Sequence, Base Caller Name, Base Caller Version, Flow Cell Barcode, Fragment Maximum Length, Fragment Mean Length, Fragment Minimum Length, Fragment Standard Deviation Length, Lane Number, Library Strand, Multiplex Barcode, Size Selection Range, Target Depth, To Trim Adapter Sequence, Transcript Integrity Number, RIN, DV200, Adapter Content, Basic Statistics, Encoding, Kmer Content, Overrepresented Sequences, Per Base N Content, Per Base Sequence Content, Per Base Sequence Quality, Per Sequence GC Content, Per Sequence Quality Score, Per Tile Sequence Quality, Percent GC Content, Sequence Duplication Levels, Sequence Length Distribution, Total Reads, QC Workflow Type, QC Workflow Version, QC Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk RNA-seq Level 2</td>
<td style="text-align: left;">Bulk RNA-seq alignment protocol description</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Url, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Contamination, Contamination Error, Mean Coverage, MSI Workflow Link, MSI Score, MSI Status, Pairs On Diff CHR, Total Reads, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Proportion Reads Mapped, Proportion Targets No Coverage, Proportion Base Mismatch, Short Reads, Is lowest level</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk RNA-seq Level 3</td>
<td style="text-align: left;">Bulk RNA-seq gene expression matrices</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Pseudo Alignment Used, Data Category, Expression Units, Matrix Type, Fusion Gene Detected, Fusion Gene Identity</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk WES Level 1</td>
<td style="text-align: left;">Bulk Whole Exome Sequencing raw files</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Sequencing Batch ID, Library Layout, Read Indicator, Library Selection Method, Read Length, Target Capture Kit, Library Preparation Kit Name, Library Preparation Kit Vendor, Library Preparation Kit Version, Sequencing Platform, Adapter Name, Adapter Sequence, Base Caller Name, Base Caller Version, Flow Cell Barcode, Fragment Maximum Length, Fragment Mean Length, Fragment Minimum Length, Fragment Standard Deviation Length, Lane Number, Multiplex Barcode, Library Preparation Days from Index, Size Selection Range, Target Depth, To Trim Adapter Sequence</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk WES Level 2</td>
<td style="text-align: left;">Bulk Whole Exome Sequencing aligned files and QC</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Contamination, Contamination Error, Mean Coverage, Adapter Content, Basic Statistics, Encoding, Overrepresented Sequences, Per Base N Content, Per Base Sequence Content, Per Base Sequence Quality, Per Sequence GC Content, Per Sequence Quality Score, Per Tile Sequence Quality, Percent GC Content, Sequence Duplication Levels, Sequence Length Distribution, QC Workflow Type, QC Workflow Version, QC Workflow Link, MSI Workflow Link, MSI Score, MSI Status, Pairs On Diff CHR, Total Reads, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Proportion Reads Mapped, Proportion Targets No Coverage, Proportion Base Mismatch, Short Reads, Proportion Coverage 10x, Proportion Coverage 30X,Is lowest level</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk WES Level 3</td>
<td style="text-align: left;">Bulk Whole Exome Sequencing called variants</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Genomic Reference, Genomic Reference URL, Germline Variants Workflow URL, Germline Variants Workflow Type, Somatic Variants Workflow URL, Somatic Variants Workflow Type, Somatic Variants Sample Type, Structural Variant Workflow URL, Structural Variant Workflow Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Level 1</td>
<td style="text-align: left;">Microarray Level 1 refers to the raw text table of probe level intensities</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Data File ID, HTAN Participant ID, HTAN Parent Biospecimen ID, Nucleic Acid Source, Microarray Platform ID, Microarray Molecule, Microarray Label, Microarray Value Definition, Microarray Protocol Auxiliary File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Level 2</td>
<td style="text-align: left;">Microarray Level 2 provides a normalized matrix of values.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID, HTAN Data File ID, Microarray Platform ID, Normalization Method</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATAC-seq Level 1</td>
<td style="text-align: left;">scATAC-seq files containing sequence read information, with or without alignment, as FASTQ or BAM files</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, Dissociation Method, Single Nucleus Buffer, Single Cell Isolation Method, Transposition Reaction, scATACseq Library Layout, Nucleus Identifier, Nuclei Barcode Length, Nuclei Barcode Read, scATACseq Read1, scATACseq Read2, scATACseq Read3, Library Construction Method, Sequencing Platform, Threshold for Minimum Passing Reads, Total Number of Passing Nuclei, Median Fraction of Reads in Peaks, Median Fraction of Reads in Annotated cis DNA Elements, Median Passing Read Percentage, Median Percentage of Mitochondrial Reads per Nucleus,Technical Replicate Group, Total Reads, Protocol Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATAC-seq Level 2</td>
<td style="text-align: left;">scATAC-seq files containing aligned sequence data, as a BAM file</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Url, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Mean Coverage, Pairs On Diff CHR, Total Reads, Proportion Reads Mapped, MapQ30, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Short Reads, Proportion Coverage 10x, Proportion Coverage 30X, Proportion Targets No Coverage, Proportion Base Mismatch, Median Percentage of Mitochondrial Reads per Nucleus, Contamination,Contamination Error</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATAC-seq Level 3</td>
<td style="text-align: left;">Processed data files containing peak information for cells</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, scATAC-seq Object ID, nCount Peaks, nFeature Peaks, Total Read-Pairs, Duplicate Read-Pairs, Chimeric Read-Pairs, Unmapped Read-Pairs, LowMapQ, Mitochondrial Read-Pairs, Passed Filters, TSS Fragments, DNase Sensitive Region Fragments, Enhancer Region Fragments, Promoter Region Fragments, On Target Fragments, Blacklist Region Fragments, Peak Region Fragments, Peak Region Cutsites, Nucleosome Signal, Nucleosome Percentile, TSS Enrichment, TSS Percentile, Pct Reads in Peaks, Blacklist Ratio, Seurat Clusters, nCount RNA, nFeature RNA, MACS2 Seqnames, MACS2 Start, MACS2 End, MACS2 Width, MACS2 Strand, MACS2 Name, MACS2 Score, MACS2 Fold Change, MACS2 Neg Log10 pvalue Summit, MACS2 Neg Log10 qvalue Summit, MACS2 Relative Summit Position</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scmC-seq Level 1</td>
<td style="text-align: left;">Files contain raw scmC-seq data.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, scmCseq Read1, scmCseq Read2, scmCseq Read3, Single Cell Isolation Method, Single Nucleus Buffer, Single Nucleus Capture, Bisulfite Conversion, Library Layout, Nucleus Identifier, Sequencing Platform, Technical Replicate Group, Median Fraction of Reads in Peaks, Median Passing Read Percentage, Peaks Calling Software, Median Percentage of Mitochondrial Reads per Nucleus, Threshold for Minimum Passing Reads, Total Number of Passing Nuclei, Total Reads</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scmC-seq Level 2</td>
<td style="text-align: left;">Files contain scmC-seq files containing aligned sequence data, as a BAM file.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Url, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Contamination, Contamination Error, Mean Coverage, Pairs On Diff CHR, Total Reads, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Proportion Reads Mapped, Proportion Targets No Coverage, Proportion Base Mismatch, Short Reads</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATAC-seq Level 4</td>
<td style="text-align: left;">Data represents the relationships between cells derived from Level 3 expression data and shown as tSNE or UMAP coordinates per cell, plus all other cell-specific meta information (e.g., cell type)</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, scATACseq Workflow Type, scATACseq Workflow Parameters Description, Workflow Version, Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scDNA-seq Level 1</td>
<td style="text-align: left;">Single-cell DNA-seq</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Sequencing Batch ID, Library Layout, Nucleic Acid Source, Library Selection Method, Read Length, Library Preparation Kit Name, Library Preparation Kit Vendor, Library Preparation Kit Version, Adapter Name, Adapter Sequence, Base Caller Name, Base Caller Version, Flow Cell Barcode, Fragment Maximum Length, Fragment Mean Length, Fragment Minimum Length, Fragment Standard Deviation Length, Lane Number, Library Strand, Multiplex Barcode, Size Selection Range, Target Depth, To Trim Adapter Sequence, Adapter Content, Basic Statistics, Encoding, Kmer Content, Overrepresented Sequences, Per Base N Content, Per Base Sequence Content, Per Base Sequence Quality, Per Sequence GC Content, Per Sequence Quality Score, Per Tile Sequence Quality, Percent GC Content, Sequence Duplication Levels, Sequence Length Distribution, Total Reads, QC Workflow Type, QC Workflow Version, QC Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scDNA-seq Level 2</td>
<td style="text-align: left;">Alignment workflows downstream of scDNA-seq Level 1</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Url, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Mean Coverage, Pairs On Diff CHR, Total Reads, Proportion Reads Mapped, MapQ30, Total Uniquely Mapped, Total Unmapped reads,Proportion Reads Duplicated, Short Reads, Proportion Coverage 10x, Proportion Coverage 30X, Proportion Targets No Coverage, Proportion Base Mismatch, Proportion Mitochondrial Reads, Contamination, Contamination Error</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multiplexed CITE-seq Level 1</td>
<td style="text-align: left;">Raw sequencing files for the multiplexed CITE-seq assay</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source,Cryopreserved Cells in Sample, Single Cell Isolation Method, Dissociation Method, Library Construction Method,Read Indicator, Read1, Read2, End Bias, Reverse Transcription Primer, Spike In, Spike In Concentration, Sequencing Platform, Total Number of Input Cells, Input Cells and Nuclei, Library Preparation Days from Index, Single Cell Dissociation Days from Index, Sequencing Library Construction Days from Index, Nucleic Acid Capture Days from Index, Protocol Link, Technical Replicate Group, Empty Well Barcode,Well Index,Feature Reference Id, Associated mRNA Library Data File ID, Single Cell Barcode Method Applied, Feature Barcode Library Type, Barcode Folder Synapse ID, Barcode Folder File List</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multiplexed CITE-seq Level 2</td>
<td style="text-align: left;">Alignment workflows downstream of Multiplexed CITE-seq Level 1</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Data File ID, Associated mRNA Library Data File ID, scRNAseq Workflow Type, Workflow Version, scRNAseq Workflow Parameters Description, Workflow Link, Genomic Reference, Genomic Reference URL, Genome Annotation URL, Checksum, Whitelist Cell Barcode File Link, Cell Barcode Tag, UMI Tag, Applied Hard Trimming</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multiplexed CITE-seq Level 3</td>
<td style="text-align: left;">Gene and Isoform expression files</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Parent Biospecimen ID, HTAN Data File ID, Associated mRNA Library Data File ID, Data Category, Matrix Type, Linked Matrices, Cell Median Number Reads, Cell Median Number Genes, Cell Total, scRNAseq Workflow Type, scRNAseq Workflow Parameters Description, Workflow Link, Workflow Version</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multiplexed CITE-seq Level 4</td>
<td style="text-align: left;">Data represents the relationships between cells derived from Level 3 expression data and shown as tSNE or UMAP coordinates per cell, plus all other cell-specific meta information (e.g., cell type)</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Parent Biospecimen ID, HTAN Data File ID, Associated mRNA Library Data File ID, scRNAseq Workflow Type, scRNAseq Workflow Parameters Description, Workflow Version, Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk Methylation-seq Level 1</td>
<td style="text-align: left;">Raw data for bulk methylation sequencing, such as FASTQs and unaligned BAMs</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, Bisulfite Conversion, Sequencing Platform, Replicate Type, Bulk Methylation Assay Type, Total DNA Input</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk Methylation-seq Level 2</td>
<td style="text-align: left;">Aligned primary data for bulk methylation sequencing, such as gene expression matrix files, VCFs, etc.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Url, Trimmer, Bulk Methylation Genomic Reference, Genomic Reference URL, Index File Name, Alignment Workflow Type, Duplicate Removal Software, Mean Coverage, Library Layout, Average Base Quality, Average Insert Size, Average Read Length, Contamination, Contamination Error, Pairs On Diff CHR, Total Reads, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Proportion Reads Mapped, Proportion Targets No Coverage, Proportion Base Mismatch, Short Reads, Proportion of Minimum CpG Coverage 10X, Proportion Coverage 30X</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bulk Methylation-seq Level 3</td>
<td style="text-align: left;">Sample level summary data for bulk methylation sequencing, such as t-SNE plot coordinates, etc.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID,DMC Calling Tool, DMC Calling Workflow URL, DMR Calling Tool, DMR Calling Workflow URL, pUC19 methylation ratio, Lambda methylation ratio, DMC data file format, DMR data file Format</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 1</td>
<td style="text-align: left;">Raw imaging data</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Imaging Assay Type, Protocol Link, Software and Version, Commit SHA, Pre-processing Completed, Pre-processing Required, Comment</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 2</td>
<td style="text-align: left;">Raw and pre-processed image data</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Data File ID, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance,WorkingDistanceUnit, Immersion, Pyramid, Zstack, Tseries, Passed QC, Comment, FOV number, FOVX, FOVXUnit, FOVY, FOVYUnit, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, PhysicalSizeZ, PhysicalSizeZUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType, MERFISH Positions File, MERFISH Codebook File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MERFISH Positions File</td>
<td style="text-align: left;">The positions file is an auxiliary MERFISH file that describes the location of bead positions in the assay.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MERFISH Codebook File</td>
<td style="text-align: left;">The codebook is an auxiliary MERFISH file that describes how each grouping of bits is converted to a gene name.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 3 Segmentation</td>
<td style="text-align: left;">Object segmentations</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID,  HTAN Data File ID, Imaging Segmentation Data Type, Parameter file, Software and Version, Commit SHA, Imaging Object Class, Number of Objects</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 3 Image</td>
<td style="text-align: left;">Quality controlled imaging data</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Parent Channel Metadata ID, HTAN Data File ID, Imaging Assay Type, Protocol Link,Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance, Immersion, Pyramid, Zstack, Tseries, Passed QC, Comment, FOV number, FOVX, FOVY, Frame Averaging</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10x Visium Spatial Transcriptomics - RNA-seq Level 1</td>
<td style="text-align: left;">Files contain raw RNA-seq data associated with spot/slide data.</td>
<td style="text-align: left;">Component, Filename, Run ID, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Read Indicator, Spatial Read1, Spatial Read2, Spatial Library Construction Method, Library Preparation Days from Index, Sequencing Library Construction Days from Index, End Bias, Reverse Transcription Primer, Sequencing Platform, Capture Area, Protocol Link, Slide Version, Slide ID, Image Re-orientation, Permeabilization Time, RIN, DV200</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10x Visium Spatial Transcriptomics - RNA-seq Level 2</td>
<td style="text-align: left;">Alignment workflows downstream of Spatial Transcriptomics RNA-seq Level 1.</td>
<td style="text-align: left;">Component, Filename, File Format, Checksum,HTAN Parent Data File ID, HTAN Data File ID, UMI Tag, Whitelist Spatial Barcode File Link, Spatial Barcode Tag, Applied Hard Trimming, Workflow Version, Workflow Link, Genomic Reference, Genomic Reference URL, Genome Annotation URL, HTAN Parent Biospecimen ID, Run ID, Capture Area</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10x Visium Spatial Transcriptomics - Auxiliary Files</td>
<td style="text-align: left;">Auxiliary data associated with spot/slide analysis (aligned Images, quality control files, etc)  from Spatial Transcriptomics.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Data File ID, Run ID, Visium File Type, Slide ID, Capture Area, Workflow Version, Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10x Visium Spatial Transcriptomics - RNA-seq Level 3</td>
<td style="text-align: left;">Processed data files based on Spatial Transcriptomics RNA-seq Level 2 and Spatial Transcriptomics Auxiliary files.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Data File ID, Run ID, Visium File Type, Workflow Version, Workflow Link, Capture Area, Spots under tissue, Mean Reads per Spatial Spot, Median Number Genes per Spatial Spot, Sequencing Saturation, Proportion Reads Mapped, Proportion Reads Mapped to Transcriptome, Median UMI Counts per Spot</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10x Visium Spatial Transcriptomics - RNA-seq Level 4</td>
<td style="text-align: left;">Processed data files based on Spatial Transcriptomics RNA-seq Level 3.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Data File ID, Run ID, Workflow Version, Workflow Link, Visium Workflow Type, Visium Workflow Parameters Description</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Visium File Type</td>
<td style="text-align: left;">The file type generated for the visium experiment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Run ID</td>
<td style="text-align: left;">A unique identifier for this individual run (typically associated with a single slide) of the spatial transcriptomic processing workflow.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Capture Area</td>
<td style="text-align: left;">Area (or Capture Area) - One of the either four or two active regions where tissue can be placed on a Visium slide. Each area is intended to contain only one tissue sample. Slide areas are named consecutively from top to bottom: A1, B1, C1, D1 for Visium slides with 6.5 mm Capture Area and A, B for CytAssist slides with 11 mm Capture Area. Both CytAssist slides with 6.5 mm Capture Area and Gateway Slides contain only two slide areas, A1 and D1.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide Version</td>
<td style="text-align: left;">Version of imaging slide used. Slide version is critical for the analysis of the sequencing data as different slides have different capture area layouts.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide ID</td>
<td style="text-align: left;">For Visium, it is the unique identifier printed on the label of each Visium slide. The serial number starts with V followed by a number which can range between one through five and ends with a dash and a three digit number, such as 123. For CosMx, this refers to the loaded Flow Cell ID. For Xenium, this ID indicates the slide orientation, as it matches the relative location of the ID on the physical Xenium slide.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Image Re-orientation</td>
<td style="text-align: left;">To ensure good fiducial alignment and tissue spots detection, it is important to correct for this shift in orientation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Permeabilization Time</td>
<td style="text-align: left;">Fixed and stained tissue sections are permeabilized for different times. Each Capture Area captures polyadenylated mRNA from the attached tissue section. Measure is provided in minutes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Whitelist Spatial Barcode File Link</td>
<td style="text-align: left;">Link to file listing all possible spatial barcodes. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Barcode Tag</td>
<td style="text-align: left;">SAM tag for spot barcode field; please provide a valid spot barcode tag (e.g. CB:Z)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Barcode Offset</td>
<td style="text-align: left;">Offset in sequence for spot barcode read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Barcode Length</td>
<td style="text-align: left;">Length of spot barcode read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Read1</td>
<td style="text-align: left;">Read 1 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Read2</td>
<td style="text-align: left;">Read 2 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Library Construction Method</td>
<td style="text-align: left;">Process which results in the creation of a library from fragments of DNA using cloning vectors or oligonucleotides with the role of adaptors [OBI_0000711]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spatial Barcode and UMI</td>
<td style="text-align: left;">Spot and transcript identifiers</td>
<td style="text-align: left;">Spatial Barcode Offset, Spatial Barcode Length, UMI Barcode Offset, UMI Barcode Length</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Mean Reads per Spatial Spot</td>
<td style="text-align: left;">The number of reads, both under and outside of tissue, divided by the number of barcodes associated with a spot under tissue.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Visium Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze the visium data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Visium Workflow Parameters Description</td>
<td style="text-align: left;">Parameters used to run the workflow..</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spots under tissue</td>
<td style="text-align: left;">The number of barcodes associated with a spot under tissue.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median UMI Counts per Spot</td>
<td style="text-align: left;">The median number of UMI counts per tissue covered spot.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Sequencing Saturation</td>
<td style="text-align: left;">The fraction of reads originating from an already-observed UMI. This is a function of library complexity and sequencing depth. More specifically, this is the fraction of confidently mapped, valid spot-barcode, valid UMI reads that had a non-unique (spot-barcode, UMI, gene).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Reads Mapped to Transcriptome</td>
<td style="text-align: left;">Fraction of reads that mapped to a unique gene in the transcriptome. The read must be consistent with annotated splice junctions. These reads are considered for UMI counting.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median Number Genes per Spatial Spot</td>
<td style="text-align: left;">The median number of genes detected per spot under tissue-associated barcode. Detection is defined as the presence of at least 1 UMI count.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">NanoString GeoMx DSP Spatial Transcriptomics Level 1</td>
<td style="text-align: left;">Files contain raw data output from the NanoString GeoMx DSP Pipeline. These can include RCC or DCC Files.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Synapse ID of GeoMx DSP PKC File, GeoMx DSP NGS Sequencing Platform, GeoMx DSP NGS Library Selection Method, GeoMx DSP NGS Library Preparation Kit Name, GeoMx DSP Library Preparation Kit Vendor, GeoMx DSP Library Preparation Kit Version, Synapse ID of GeoMx Lab Worksheet File, Software and Version</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Assay Type</td>
<td style="text-align: left;">The assay type which was used for the GeoMx DSP pipeline.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Synapse ID of GeoMx DSP PKC File</td>
<td style="text-align: left;">The Synapse ID(s) associated with the PKC mapping file for the assay. Multiple files are listed as comma separated values.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP NGS Sequencing Platform</td>
<td style="text-align: left;">A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP NGS Library Selection Method</td>
<td style="text-align: left;">How RNA molecules are isolated.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP NGS Library Preparation Kit Name</td>
<td style="text-align: left;">Name of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Library Preparation Kit Vendor</td>
<td style="text-align: left;">Vendor of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Library Preparation Kit Version</td>
<td style="text-align: left;">Version of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Synapse ID of GeoMx Lab Worksheet File</td>
<td style="text-align: left;">Synapse ID(s) of Lab Worksheet Files output from the GeoMx DSP workflow. Multiple files are listed as comma separated values.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NanoString GeoMx DSP Spatial Transcriptomics Level 3</td>
<td style="text-align: left;">Files contain processed data from the NanoString GeoMx DSP Pipeline. This level depends on GeoMx Level 1 and Imaging Level 2.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, GeoMx DSP Assay Type, Synapse ID of GeoMx DSP ROI Segment Annotation File, GeoMx DSP Unique Probe Count, GeoMx DSP Unique Target Count, GeoMx DSP Genomic Reference, Matrix Type, GeoMx DSP Workflow Type, GeoMx DSP Workflow Parameter Description, GeoMx DSP Workflow Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Synapse ID of GeoMx DSP ROI Segment Annotation File</td>
<td style="text-align: left;">Synapse ID(s) for ROI/Segmentation annotations in the GeoMx DSP experiment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Genomic Reference</td>
<td style="text-align: left;">Exact version of the human genome reference used in the alignment of reads (e.g. https://www.gencodegenes.org/human/). Only applicable to some applications in GeoMx</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Unique Probe Count</td>
<td style="text-align: left;">Total number of unique probes reported.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Unique Target Count</td>
<td style="text-align: left;">Total number of unique genes reported.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze the GeoMx DSP data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Workflow Parameter Description</td>
<td style="text-align: left;">Parameters used to run the GeoMx DSP workflow.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">GeoMx DSP Workflow Link</td>
<td style="text-align: left;">Link to workflow or command. DockStore.org recommended. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NanoString GeoMx DSP ROI RCC Segment Annotation Metadata</td>
<td style="text-align: left;">GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates.</td>
<td style="text-align: left;">HTAN Parent Biospecimen ID, Scan name, ROI name, Segment name, ROI X Coordinate,ROI Y Coordinate, Tags, QC status, Scan Height, Scan Width, Scan Offset X, Scan Offset Y, Binding Density, Positive norm factor, Surface area, Nuclei count, Tissue Stain</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Scan name</td>
<td style="text-align: left;">GeoMx Scan name (as appears in Segment Summary)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">ROI name</td>
<td style="text-align: left;">ROI name (application generated). For Xenium this is referred to as the “region name”</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Segment name</td>
<td style="text-align: left;">Name given to segment at time of generation</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tags</td>
<td style="text-align: left;">Unique descriptor of a variable group (ie. MAPK+)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">ROI X Coordinate</td>
<td style="text-align: left;">X location within the image</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">ROI Y Coordinate</td>
<td style="text-align: left;">Y location within the image</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">QC status</td>
<td style="text-align: left;">ROI quality control flag as reported by the application</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Scan Height</td>
<td style="text-align: left;">Height of the scan for GeoMx Analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Scan Width</td>
<td style="text-align: left;">Width of the scan for GeoMx Analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Scan Offset X</td>
<td style="text-align: left;">Offset X of the scan for GeoMx Analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Scan Offset Y</td>
<td style="text-align: left;">Offset Y of the scan for GeoMx Analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Binding Density</td>
<td style="text-align: left;">The binding density as reported by the application</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Positive norm factor</td>
<td style="text-align: left;">The Positive Control Normalization factor calculated using pos-hyb controls</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Surface area</td>
<td style="text-align: left;">Surface area of the ROI in square microns (µm^2). In CosMx, this is referred to as the Scan Area. In Xenium, this is referred to as the Region Area</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nuclei count</td>
<td style="text-align: left;">Number of nuclei detected in the segment (if applicable)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tissue Stain</td>
<td style="text-align: left;">e.g. CD45 or PanCK (if masking was performed)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NanoString GeoMx DSP ROI DCC Segment Annotation Metadata</td>
<td style="text-align: left;">GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates.</td>
<td style="text-align: left;">HTAN Parent Biospecimen ID, Scan name, Slide name, ROI name, Segment name, ROI X Coordinate,ROI Y Coordinate, Tags, Scan Height, Scan Width, Scan Offset X, Scan Offset Y, Surface area, Nuclei count, Sequencing Saturation, MapQ30, Raw reads, Stitched reads, Aligned reads, Deduplicated reads, In Situ Negative median, Biological probe median</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide name</td>
<td style="text-align: left;">Similar to a Run ID, the slide name indicates the slide a given ROI is linked to (as reported in Segment Summary).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Raw reads</td>
<td style="text-align: left;">Reads not yet analyzed in any way to be used for data analysis. The number of reads that pass filter from the flow cell represented in the FASTQ file.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Stitched reads</td>
<td style="text-align: left;">Represents consensus from the overlapping sequence of read 1 and 2. This is a % of the aligned reads that were overlapped and consensus confirmed, usually upward of 80% but less in terms of number of reads than aligned reads</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Aligned reads</td>
<td style="text-align: left;">Is a sequence that has been aligned to a gene/probe. Typically these reads can number from the hundreds of thousands to tens of millions. In GeoMx alignment is via mapping the RTS ID to a white list of sequences that represent targets.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Deduplicated reads</td>
<td style="text-align: left;">Is the replacement of blocks of duplicate data with a Virtual Index Pointer linking the new sub-block to the existing block of data in a duplicate repository. This is used to reduce the amount of space need to store the data.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">In Situ Negative median</td>
<td style="text-align: left;">Is the median of all negative control probes for a given segment. A measure of signal to background for each segment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biological probe median</td>
<td style="text-align: left;">Is the median count from all probes except the negative control probes. A measure of signal to background for each segment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HI-C-seq Level 1</td>
<td style="text-align: left;">Unaligned sequence data</td>
<td style="text-align: left;">Component, HTAN Parent Biospecimen ID, HTAN Data File ID, Filename, File Format, Genomic Reference, Sequencing Platform, Nucleic Acid Source, Technical Replicate Group, Transposition Reaction, Crosslinking Condtion, DNA Digestion Condition, Nuclei Permeabilization Method, Ligation Condition, Biotin Enrichment, DNA Input Amount, Total Reads, Protocol Link</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HI-C-seq Level 2</td>
<td style="text-align: left;">Aligned read pairs, contact matrix</td>
<td style="text-align: left;">Component, HTAN Data File ID, HTAN Parent Data File ID, Filename, File Format, Genomic Reference, Aligned Read Length, Tool, Resolution, Normalization Method</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HI-C-seq Level 3</td>
<td style="text-align: left;">Summary data for the HI-C-seq assay.</td>
<td style="text-align: left;">Component, HTAN Parent Data File ID, HTAN Data File ID, Filename, File Format, Genomic Reference, Stripe Calling, Loop Window, Stripe Window, Loop Calling</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Crosslinking Condtion</td>
<td style="text-align: left;">Detailed condition for DNA crosslinking</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DNA Digestion Condition</td>
<td style="text-align: left;">Enzymes and treatment length/temperature for genome digestion</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nuclei Permeabilization Method</td>
<td style="text-align: left;">Detergent and treatment condition for nuclei permeabilization and crosslinking softening</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Ligation Condition</td>
<td style="text-align: left;">Name of ligase and condition for proximity ligation</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Biotin Enrichment</td>
<td style="text-align: left;">Whether biotin is used for enriching ligation product</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DNA Input Amount</td>
<td style="text-align: left;">Amount of DNA for library construction, in nanograms.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Resolution</td>
<td style="text-align: left;">Binning size used for generating contact matrix, in basepair.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Stripe Calling</td>
<td style="text-align: left;">Tool used for identifying architectural stripe-forming, interaction hotspots.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Loop Window</td>
<td style="text-align: left;">Binning size used for calling significant dot interactions (loops)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Stripe Window</td>
<td style="text-align: left;">Binning size used for calling significant architectural stripes. Can be an integer or comma-separated list of integers indicating bin size and sliding window size if different.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Loop Calling</td>
<td style="text-align: left;">Tool used for identifying loop interactions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 4</td>
<td style="text-align: left;">Derived imaging data: Object-by-feature array</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Parent Channel Metadata ID, HTAN Data File ID, Parameter file, Software and Version, Commit SHA,Number of Objects, Number of Features,Imaging Object Class, Imaging Summary Statistic</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">SRRS Imaging Level 2</td>
<td style="text-align: left;">SRRS-specific HTAN raw and pre-processed image data</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Data File ID, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, Pyramid, Zstack, Tseries, Passed QC, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">10X Genomics Xenium ISS Experiment</td>
<td style="text-align: left;">All data pertaining to the 10X Genomics Xenium In-Situ Hybridization experiment</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Xenium Bundle Contents, Slide ID, ROI name, Panel Name, Protocol Link, Software and Version,Total Number of Cells, Total Number of Targets, Surface area, Experiment IF Channels, Transcripts per Cell, Percent of Transcripts within Cells, Decoded Transcripts, Xenium IF image HTAN File ID, Xenium HE image HTAN File ID</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Xenium Bundle Contents</td>
<td style="text-align: left;">A comma separated list of filenames within the Xenium bundle zip file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Panel Name</td>
<td style="text-align: left;">The human-readable panel name. This could be the Gene Panel name or Protein Panel name. In Xenium, this refers to the string entered as the name in panel specification (e.g. Xenium Human Immuno-Oncology Add-on B Gene Expression). In CosMx, this refers to the panel name as it appears in the CosMx catalog (e.g. CosMx Human Universal Cell Characterization Panel (1000-plex))</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Total Number of Cells</td>
<td style="text-align: left;">The total number of cells analyzed on the flow cell</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Total Number of Targets</td>
<td style="text-align: left;">Refers to the target of an assay. Can be genes/transcripts or probes</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Experiment IF Channels</td>
<td style="text-align: left;">A comma-separated list with any number of channels the user deems appropriate(Example: PanCK, CD45, CD3, DAPI)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Transcripts per Cell</td>
<td style="text-align: left;">Mean or Median transcript count per cell analyzed on the flow cell or slide</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Percent of Transcripts within Cells</td>
<td style="text-align: left;">The percentage of transcripts assigned to assayed cells</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Decoded Transcripts</td>
<td style="text-align: left;">In Xenium, this is the number of high-quality, decoded-to-gene nuclear transcripts divided by the total segmented nuclear area to get a transcript density (units are reported in 100um^2).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Xenium IF image HTAN File ID</td>
<td style="text-align: left;">The HTAN Data File ID of a Imaging Level 2 file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Xenium HE image HTAN File ID</td>
<td style="text-align: left;">The HTAN Data File ID of a Imaging Level 2 file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">RPPA Level 2</td>
<td style="text-align: left;">Array based protemics. Each dilution curve of spot intensities is fitted using the monotone increasing B-spline model in the SuperCurve R package. This fits a single curve using all the samples on a slide with the signal intensity as the response variable and the dilution steps as independent variables. The fitted curve is plotted with the signal intensities on the y-axis and the log2-concentration of proteins on the x-axis for diagnostic purposes.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID, HTAN Data File ID, HTAN RPPA Antibody Table, Assay Type, Protocol Link, Software and Version</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HTAN RPPA Antibody Table</td>
<td style="text-align: left;">A table containing antibody level metadata for RPPA</td>
<td style="text-align: left;">HTAN RPPA Antibody Table ID, Filename, File Format, Ab Name Reported on Dataset, GENCODE Gene Symbol Target, UNIPROT Protein ID Target, Phosphoprotein Flag, Vendor, Catalog Number, Internal Ab ID, Species, RPPA Dilution, Phospho Site, RPPA Validation Status, Clone, Clonality, Antibody Notes</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">RPPA Level 3</td>
<td style="text-align: left;">Level 3 Reverse Phase Protein Array (RPPA) data contains intra-batch normalized intensities.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID, HTAN Data File ID, Assay Type, Software and Version, Normalization Method</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">RPPA Level 4</td>
<td style="text-align: left;">Level 4 Reverse Phase Protein Array (RPPA) data contains intra-batch corrected intensities.</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Data File ID, Assay Type, Batch Correction Method</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Nanostring CosMx SMI Experiment</td>
<td style="text-align: left;">RNA and Protein Panel assays applied as part of Nanostring CosMx Spatial Molecular Imager (SMI)</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, CosMx Bundle Contents, Slide ID, CosMx Assay Type, Panel Name, Protocol Link, Software and Version, Total Number of Cells, Total Number of Targets, Number of FOVs, Surface area, Experiment IF Channels, Transcripts per Cell, Percent of Transcripts within Cells, Mean Total Transcripts per Area, Unique Genes, Total Negative Probe Counts</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CosMx Bundle Contents</td>
<td style="text-align: left;">A comma separated list of filenames within the CosMx bundle zip file</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CosMx Assay Type</td>
<td style="text-align: left;">The specification for barcodes on each image. Either RNA probe or protein antibody according to the assay</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Number of FOVs</td>
<td style="text-align: left;">The total number of FOVs recorded for the sample on a single flow cell</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Mean Total Transcripts per Area</td>
<td style="text-align: left;">The mean total transcripts per um3</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Unique Genes</td>
<td style="text-align: left;">The total unique genes detected above background</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Negative Probe Counts</td>
<td style="text-align: left;">Mean Total Negative probe counts/cell</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Mass Spectrometry Level 1</td>
<td style="text-align: left;">Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics, level 1</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, MS Batch ID, MS-based Assay Type, Analyte Type, MS-based Targeted, MS Instrument Vendor and Model, MS Source, Polarity, Mass Range Low Value, Mass Range High Value, Data Collection Mode, MS Scan Mode, MS Labeling, Protocol Link, LC Instrument Vendor and Model, LC Column Vendor and Model, LC Resin, LC Length Value, LC Temp Value, LC ID Value, LC Flow Rate, LC Gradient, LC Mobile Phase A, LC Mobile Phase B, Software and Version, MS Instrument Metadata File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mass Spectrometry Level 2</td>
<td style="text-align: left;">Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics, level 2</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, MS Assay Category, Software and Version, Mass Spectrometry Auxiliary File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mass Spectrometry Level 3</td>
<td style="text-align: left;">Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics, level 3</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, MS Assay Category, Software and Version, Mass Spectrometry Auxiliary File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mass Spectrometry Level 4</td>
<td style="text-align: left;">Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics, level 4</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, MS Assay Category, Software and Version, Mass Spectrometry Auxiliary File</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mass Spectrometry Auxiliary File</td>
<td style="text-align: left;">Auxiliary software parameter file used in mass spectrometry data processing, recorded as synapse ID (syn12345).</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Level 3 Channels</td>
<td style="text-align: left;">Channel-level Metadata Attributes</td>
<td style="text-align: left;">HTAN Channel Metadata ID, Channel ID, Channel Name, Channel Passed QC, Cycle Number, Sub Cycle Number, Antibody Role, Target Name, Antibody Name, RRID identifier, Fluorophore, Clone, Lot, Vendor, Catalog Number, Excitation Wavelength, Emission Wavelength, Excitation Bandwidth, Emission Bandwidth, Metal Isotope Element, Metal Isotope Mass, Oligo Barcode Upper Strand, Oligo Barcode Lower Strand, Dilution, Concentration</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Channel Metadata ID</td>
<td style="text-align: left;">HTAN ID for this channel metadata table (same for all rows)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Channel ID</td>
<td style="text-align: left;">This must match the corresponding field in the OME-XML / TIFF header. (eg 'Channel:0:1')</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Channel Name</td>
<td style="text-align: left;">This must match the corresponding field in the OME-XML / TIFF header. (eg 'Blue' or 'CD45' or 'E-cadherin')</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Channel Passed QC</td>
<td style="text-align: left;">Identify stains that did not pass QC but are included in the dataset.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">No - Channel Failed QC</td>
<td style="text-align: left;">Channel failed QC</td>
<td style="text-align: left;">Channel QC Failure Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Channel QC Failure Type</td>
<td style="text-align: left;">Reason the channel failed QC</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other/multiple channel QC faliure types</td>
<td style="text-align: left;">QC failure type not speficied</td>
<td style="text-align: left;">Channel QC Failure Comment</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Channel QC Failure Comment</td>
<td style="text-align: left;">Custom comment on channel QC faliure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cycle Number</td>
<td style="text-align: left;">The cycle # in which the co-listed reagent(s) was(were) used. Integer &gt;= 1 (up to number of cycles)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sub Cycle Number</td>
<td style="text-align: left;">Sub cycle number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Target Name</td>
<td style="text-align: left;">Short descriptive name (abbreviation) for this target (antigen)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Antibody Role</td>
<td style="text-align: left;">"Is this antibody acting as a primary or secondary antibody"</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Antibody Name</td>
<td style="text-align: left;">Antibody Name (free text (eg “Keratin”, “CD163”, “DNA”))</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">RRID identifier</td>
<td style="text-align: left;">Research Resource Identifier (eg “RRID: AB_394606”)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Fluorophore</td>
<td style="text-align: left;">Fluorescent dye label (eg Alexa Fluor 488)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Clone</td>
<td style="text-align: left;">Clone</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lot</td>
<td style="text-align: left;">Lot number from vendor</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Vendor</td>
<td style="text-align: left;">Vendor</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Catalog Number</td>
<td style="text-align: left;">Catalog Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Excitation Wavelength</td>
<td style="text-align: left;">Center/peak of the excitation spectrum (nm)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Emission Wavelength</td>
<td style="text-align: left;">Center/peak of the emission spectrum (nm)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Excitation Bandwidth</td>
<td style="text-align: left;">Nominal width of excitation spectrum (nm)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Emission Bandwidth</td>
<td style="text-align: left;">Nominal width of emission spectrum (nm)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Metal Isotope Element</td>
<td style="text-align: left;">Element abbreviation. eg “La” or “Nd”</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Metal Isotope Mass</td>
<td style="text-align: left;">Element mass number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Oligo Barcode Upper Strand</td>
<td style="text-align: left;">Oligo Barcode - Upper Strand</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Oligo Barcode Lower Strand</td>
<td style="text-align: left;">Oligo Barcode - Lower Strand</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Dilution</td>
<td style="text-align: left;">Dilution (eg 1:1000)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Concentration</td>
<td style="text-align: left;">Concentration (eg 10ug/mL)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Assay Type</td>
<td style="text-align: left;">Type of imaging assay</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Channel Metadata Filename</td>
<td style="text-align: left;">Full path within Synapse project of uploaded companion CSV file containing channel-level metadata details</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microscope</td>
<td style="text-align: left;">Microscope type (manufacturer, model, etc) used for this experiment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Objective</td>
<td style="text-align: left;">Objective</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NominalMagnification</td>
<td style="text-align: left;">The magnification of the lens as specified by the manufacturer - i.e. '60' is a 60X lens. floating point value &gt; 1(no units)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LensNA</td>
<td style="text-align: left;">The numerical aperture of the lens. Floating point value &gt; 0.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">WorkingDistance</td>
<td style="text-align: left;">The working distance of the lens, expressed as a floating point number. Floating point &gt; 0.</td>
<td style="text-align: left;">WorkingDistanceUnit</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">WorkingDistanceUnit</td>
<td style="text-align: left;">The units of the working distance. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immersion</td>
<td style="text-align: left;">Immersion medium</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pyramid</td>
<td style="text-align: left;">Does data file contain pyramid of images</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Zstack</td>
<td style="text-align: left;">Does data file contain a Z-stack of images</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tseries</td>
<td style="text-align: left;">Does data file contain a time-series of images</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Passed QC</td>
<td style="text-align: left;">Did all channels pass QC (if not add free text Comment)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">No - Channels QC</td>
<td style="text-align: left;">Not all channels passed QC</td>
<td style="text-align: left;">Comment</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Comment</td>
<td style="text-align: left;">Free text field (generally for QC comment)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FOV number</td>
<td style="text-align: left;">Index of FOV (as it pertains to its sequence order). Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FOVX</td>
<td style="text-align: left;">Field of view X dimension. Floating point</td>
<td style="text-align: left;">FOVXUnit</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FOVXUnit</td>
<td style="text-align: left;">Field of view X dimension units. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FOVY</td>
<td style="text-align: left;">Field  of view Y dimension. Floating point value</td>
<td style="text-align: left;">FOVYUnit</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FOVYUnit</td>
<td style="text-align: left;">Field of view Y dimension units. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Frame Averaging</td>
<td style="text-align: left;">Number of frames averaged together (if no averaging, set to 1). Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Image ID</td>
<td style="text-align: left;">Unique internal image identifier. eg "Image:0". (To be extracted from OME-XML)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DimensionOrder</td>
<td style="text-align: left;">The order in which the individual planes of data are interleaved.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeX</td>
<td style="text-align: left;">Physical size (X-dimension) of a pixel. Units are set by PhysicalSizeXUnit. Floating point value &gt; 0.</td>
<td style="text-align: left;">PhysicalSizeXUnit</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeXUnit</td>
<td style="text-align: left;">The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeY</td>
<td style="text-align: left;">Physical size (Y-dimension) of a pixel. Units are set by PhysicalSizeYUnit. Floating point value &gt; 0.</td>
<td style="text-align: left;">PhysicalSizeYUnit</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeYUnit</td>
<td style="text-align: left;">The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeZ</td>
<td style="text-align: left;">Physical size (Z-dimension) of a pixel. Units are set by PhysicalSizeZUnit. Floating point value &gt; 0.</td>
<td style="text-align: left;">PhysicalSizeZUnit</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PhysicalSizeZUnit</td>
<td style="text-align: left;">The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Pixels BigEndian</td>
<td style="text-align: left;">Boolean (True/False)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PlaneCount</td>
<td style="text-align: left;">Number of Z-planes (not to be confused with downsampled "pyramid"). Integer &gt;=1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">SizeC</td>
<td style="text-align: left;">Number of channels. Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">SizeT</td>
<td style="text-align: left;">Number of time points. Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">SizeX</td>
<td style="text-align: left;">Size of image: X dimension (in pixels). Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">SizeY</td>
<td style="text-align: left;">Size of image: Y dimension (in pixels). Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">SizeZ</td>
<td style="text-align: left;">Size of image: Z dimension (in pixels). Integer &gt;= 1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">PixelType</td>
<td style="text-align: left;">Data type for each pixel value. E.g. "uint16"</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Segmentation Data Type</td>
<td style="text-align: left;">Specifies how the segmentation is stored</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Parameter file</td>
<td style="text-align: left;">Path in Syanpse to a text file listing algorithm version numbers and relevant parameters needed to reproduce the analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Commit SHA</td>
<td style="text-align: left;">Short SHA for software version [8 hexadecimal characters (for github), comma separated if multiple]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Object Class</td>
<td style="text-align: left;">Defines the structure that the mask delineates</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Object Class Other</td>
<td style="text-align: left;">Imaging Object Class Other</td>
<td style="text-align: left;">Imaging Object Class Description</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Object Class Description</td>
<td style="text-align: left;">Free text description of object class [string]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Number of Objects</td>
<td style="text-align: left;">The number of objects (eg cells) described</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Number of Features</td>
<td style="text-align: left;">The number of features (eg channels) described</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Summary Statistic</td>
<td style="text-align: left;">Function used to summarize object/feature intensity</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Nucleic Acid Source</td>
<td style="text-align: left;">The source of the input nucleic molecule</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Micro-region Seq Platform</td>
<td style="text-align: left;">The platform used for micro-regional RNA sequencing (if applicable)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">ROI Tag</td>
<td style="text-align: left;">The tag or grouping used to identify the ROI in micro-regional RNA sequencing (if applicable). Must match the ROI tag within the count matrix in level 3.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Single Cell Isolation Method</td>
<td style="text-align: left;">The method by which cells are isolated into individual reaction containers at a single cell resolution (e.g. wells, micro-droplets)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Dissociation Method</td>
<td style="text-align: left;">The tissue dissociation method used for scRNASeq or scATAC-seq assays</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Layout</td>
<td style="text-align: left;">Sequencing read type</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nucleus Identifier</td>
<td style="text-align: left;">Unique nuclei barcode; added at transposition step. Determines which nucleus the reads originated from</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nuclei Barcode</td>
<td style="text-align: left;">Nuclei Barcode</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Library Layout</td>
<td style="text-align: left;">Sequencing read type</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nuclei Barcode Read</td>
<td style="text-align: left;">Nuclei Barcode Read</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nuclei Barcode Length</td>
<td style="text-align: left;">Nuclei Barcode Length</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Paired End</td>
<td style="text-align: left;">A library layout type</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Read1</td>
<td style="text-align: left;">Read 1 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Read2</td>
<td style="text-align: left;">Read 2 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Read3</td>
<td style="text-align: left;">Read 3 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scmCseq Read1</td>
<td style="text-align: left;">Read 1 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scmCseq Read2</td>
<td style="text-align: left;">Read 2 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scmCseq Read3</td>
<td style="text-align: left;">Read 3 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Threshold for Minimum Passing Reads</td>
<td style="text-align: left;">Threshold for calling cells</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Total Number of Passing Nuclei</td>
<td style="text-align: left;">Number of nuclei sequenced</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median Fraction of Reads in Peaks</td>
<td style="text-align: left;">Median fraction of reads in peaks (FRIP)</td>
<td style="text-align: left;">Peaks Calling Software</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median Fraction of Reads in Annotated cis DNA Elements</td>
<td style="text-align: left;">Median fraction of reads in annotated cis-DNA elements (FRIADE)</td>
<td style="text-align: left;">Peaks Calling Software</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median Passing Read Percentage</td>
<td style="text-align: left;">Non-PCR duplicate nuclear genomic sequence reads not aligning to unanchored contigs out of total reads assigned to the nucleus barcode</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median Percentage of Mitochondrial Reads per Nucleus</td>
<td style="text-align: left;">Contamination from mitochondrial sequences</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Peaks Calling Software</td>
<td style="text-align: left;">Generic name of peaks calling tool</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Read Indicator</td>
<td style="text-align: left;">Indicate if this is Read 1 (R1), Read 2 (R2), Index Reads 1 (I1), Index Reads 2 (I2),  or Other</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Read1</td>
<td style="text-align: left;">Read 1 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Read2</td>
<td style="text-align: left;">Read 2 content description</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">cDNA</td>
<td style="text-align: left;">Complementary DNA. A DNA copy of an mRNA or complex sample of mRNAs, made using reverse transcriptase</td>
<td style="text-align: left;">cDNA Offset, cDNA Length</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">cDNA Offset</td>
<td style="text-align: left;">Offset in sequence for cDNA read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">cDNA Length</td>
<td style="text-align: left;">Length of cDNA read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Barcode and UMI</td>
<td style="text-align: left;">Cell and transcript identifiers</td>
<td style="text-align: left;">UMI Barcode Offset, UMI Barcode Length, Median UMIs per Cell Number, Cell Barcode Offset, Cell Barcode Length, Valid Barcodes Cell Number</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cell Barcode Offset</td>
<td style="text-align: left;">Offset in sequence for cell barcode read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Barcode Length</td>
<td style="text-align: left;">Length of cell barcode read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Valid Barcodes Cell Number</td>
<td style="text-align: left;">Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">UMI Barcode Offset</td>
<td style="text-align: left;">Start position of UMI barcode in the sequence. Values: number, 0 for start of read</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">UMI Barcode Length</td>
<td style="text-align: left;">Length of UMI barcode read (in bp): number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Median UMIs per Cell Number</td>
<td style="text-align: left;">Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Median Number Reads</td>
<td style="text-align: left;">Median number of reads per cell. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Median Number Genes</td>
<td style="text-align: left;">Median number of genes detected per cell. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Total</td>
<td style="text-align: left;">Number of sequenced  cells. Applies to raw counts matrix only.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Construction Method</td>
<td style="text-align: left;">Process which results in the creation of a library from fragments of DNA using cloning vectors or oligonucleotides with the role of adaptors [OBI_0000711]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Input Cells and Nuclei</td>
<td style="text-align: left;">Number of cells and number of nuclei input; entry format: number, number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">CEL-seq2</td>
<td style="text-align: left;">Highly-multiplexed plate-based single-cell RNA-Seq assay</td>
<td style="text-align: left;">Empty Well Barcode, Well Index</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Empty Well Barcode</td>
<td style="text-align: left;">Unique cell barcode assigned to empty cells used as controls in CEL-seq2 assays.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Well Index</td>
<td style="text-align: left;">Indicate if protein expression (EPCAM/CD45) positive/negative data  is available for each cell in CEL-seq2 assays</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Library Preparation Days from Index</td>
<td style="text-align: left;">Number of days between sample for assay was received in lab and the libraries were prepared for sequencing [number]. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Single Cell Dissociation Days from Index</td>
<td style="text-align: left;">Number of days between sample for single cell assay was received in lab and when the sample was dissociated and cells were isolated [number]. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Sequencing Library Construction Days from Index</td>
<td style="text-align: left;">Number of days between sample for assay was received in lab and day of sequencing library construction [number]. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Nucleic Acid Capture Days from Index</td>
<td style="text-align: left;">Number of days between sample for single cell assay was received in lab and day of nucleic acid capture part of library construction (in number of days since sample received in lab) [number]. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cryopreserved Cells in Sample</td>
<td style="text-align: left;">Indicate if library preparation was based on revived frozen cells.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">End Bias</td>
<td style="text-align: left;">The end of the cDNA molecule that is preferentially sequenced, e.g. 3/5 prime tag/end or the full length transcript</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Reverse Transcription Primer</td>
<td style="text-align: left;">An oligo to which new deoxyribonucleotides can be added by DNA polymerase [SO_0000112]. The type of primer used for reverse transcription, e.g. oligo-dT or random primer. This allows users to identify content of the cDNA library input e.g. enriched for mRNA</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Feature barcoding</td>
<td style="text-align: left;">A method for adding extra channels of information to cells by running single-cell gene expression in parallel with other assays [https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/feature-bc]</td>
<td style="text-align: left;">Feature Reference Id</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Feature Reference Id</td>
<td style="text-align: left;">Unique ID for this feature. Must not contain whitespace, quote or comma characters. Each ID must be unique and must not collide with a gene identifier from the transcriptome [https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis#feature-ref]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Spike In</td>
<td style="text-align: left;">A set of known synthetic RNA molecules with known sequence that are added to the cell lysis mix</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">ERCC</td>
<td style="text-align: left;">The External RNA Controls Consortium (ERCC) spike in set is commonly used in single-cell experiments for normalization</td>
<td style="text-align: left;">Spike In Concentration</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Spike In Concentration</td>
<td style="text-align: left;">The final concentration or dilution (for commercial sets) of the spike in mix [PMID:21816910]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Sequencing Platform</td>
<td style="text-align: left;">A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Technical Replicate Group</td>
<td style="text-align: left;">A common term for all files belonging to the same cell or library. Provide a numbering of each library prep batch (can differ from encapsulation and sequencing batch)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Number of Input Cells</td>
<td style="text-align: left;">Number of cells loaded/placed on plates</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Sequencing Batch ID</td>
<td style="text-align: left;">Links samples to a specific local sequencer run. Can be string or 'null'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Single Nucleus Buffer</td>
<td style="text-align: left;">Nuclei isolation buffer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Transposition Reaction</td>
<td style="text-align: left;">Name of the transposase, transposon sequences</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Read Length</td>
<td style="text-align: left;">The length of the sequencing reads. Can be integer, null</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Target Capture Kit</td>
<td style="text-align: left;">Description that can uniquely identify a target capture kit. Suggested value is a combination of vendor, kit name, and kit version.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Selection Method</td>
<td style="text-align: left;">How RNA molecules are isolated.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Preparation Kit Name</td>
<td style="text-align: left;">Name of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Preparation Kit Vendor</td>
<td style="text-align: left;">Vendor of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Library Preparation Kit Version</td>
<td style="text-align: left;">Version of Library Preparation Kit. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Name</td>
<td style="text-align: left;">Name of the sequencing adapter. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Sequence</td>
<td style="text-align: left;">Base sequence of the sequencing adapter. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Base Caller Name</td>
<td style="text-align: left;">Name of the base caller. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Base Caller Version</td>
<td style="text-align: left;">Version of the base caller. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Flow Cell Barcode</td>
<td style="text-align: left;">Flow cell barcode. Wrong or missing information may affect analysis results. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fragment Maximum Length</td>
<td style="text-align: left;">Maximum length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fragment Mean Length</td>
<td style="text-align: left;">Mean length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fragment Minimum Length</td>
<td style="text-align: left;">Minimum length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fragment Standard Deviation Length</td>
<td style="text-align: left;">Standard deviation of the sequenced fragments length (e.g., as predicted by Agilent Bioanalyzer). Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lane Number</td>
<td style="text-align: left;">The basic machine unit for sequencing. For Illumina machines, this reflects the physical lane number. Wrong or missing information may affect analysis results. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Library Strand</td>
<td style="text-align: left;">Library stranded-ness.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multiplex Barcode</td>
<td style="text-align: left;">The barcode/index sequence used. Wrong or missing information may affect analysis results. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Size Selection Range</td>
<td style="text-align: left;">Range of size selection. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Target Depth</td>
<td style="text-align: left;">The targeted read depth prior to sequencing. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">To Trim Adapter Sequence</td>
<td style="text-align: left;">Does the user suggest adapter trimming?</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Trim Adapter Sequence</td>
<td style="text-align: left;">Trim adapter sequence</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Trimmer Name</td>
<td style="text-align: left;">Name of adapter trimmer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Trimmer Version</td>
<td style="text-align: left;">Version of the adapter trimmer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Trimmer Options</td>
<td style="text-align: left;">Options used by adapter trimmer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Transcript Integrity Number</td>
<td style="text-align: left;">Used to describe the quality of the starting material, esp. in regards to FFPE samples. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">RIN</td>
<td style="text-align: left;">A numerical assessment of the integrity of RNA based on the entire electrophoretic trace of the RNA sample including the presence or absence of degradation products. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">DV200</td>
<td style="text-align: left;">Represents the percentage of RNA fragments that are &gt;200 nucleotides in size. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adapter Content</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Basic Statistics</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Encoding</td>
<td style="text-align: left;">Version of ASCII encoding of quality values found in the file. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Kmer Content</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Overrepresented Sequences</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Base N Content</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Base Sequence Content</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Base Sequence Quality</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Sequence GC Content</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Sequence Quality Score</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Per Tile Sequence Quality</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent GC Content</td>
<td style="text-align: left;">The overall %GC of all bases in all sequences. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sequence Duplication Levels</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sequence Length Distribution</td>
<td style="text-align: left;">State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Reads</td>
<td style="text-align: left;">Total number of reads per sample. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Whitelist Cell Barcode File Link</td>
<td style="text-align: left;">Link to file listing all possible cell barcodes. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cell Barcode Tag</td>
<td style="text-align: left;">SAM tag for cell barcode field; please provide a valid cell barcode tag (e.g. CB:Z)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">UMI Tag</td>
<td style="text-align: left;">SAM tag for the UMI field; please provide a valid UB, UMI (e.g. UB:Z or UR:Z)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Applied Hard Trimming</td>
<td style="text-align: left;">Was Hard Trimming applied</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Applied Hard Trimming</td>
<td style="text-align: left;">Hard Trimming was applied</td>
<td style="text-align: left;">Aligned Read Length</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Aligned Read Length</td>
<td style="text-align: left;">Read length used for alignment if hard trimming was applied</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scRNAseq Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Workflow Version</td>
<td style="text-align: left;">Major version of the workflow (e.g. Cell Ranger v3.1)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scRNAseq Workflow Parameters Description</td>
<td style="text-align: left;">Parameters used to run the workflow. scRNA-seq level 3: e.g. Normalization and log transformation, ran empty drops or doublet detection, used filter on # genes/cell, etc. scRNA-seq Level 4: dimensionality reduction with PCA and 50 components, nearest-neighbor graph with k = 20 and Leiden clustering with resolution = 1, UMAP visualization using 50 PCA components, marker genes used to annotate cell types, information about droplet matrix (all barcodes) to cell matrix (only informative barcodes representing real cells) conversion</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">scATACseq Workflow Parameters Description</td>
<td style="text-align: left;">Parameters used to run the scATAC-seq workflow.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Workflow Link</td>
<td style="text-align: left;">Link to workflow or command. DockStore.org recommended. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">QC Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">QC Workflow Version</td>
<td style="text-align: left;">Major version for a workflow. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">QC Workflow Link</td>
<td style="text-align: left;">Link to workflow used. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Germline Variants Workflow URL</td>
<td style="text-align: left;">Link to workflow document, e.g. Github, DockStore.org recommended</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Germline Variants Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Germline Variants Workflow Type</td>
<td style="text-align: left;">Other Germline Variants Workflow Type</td>
<td style="text-align: left;">Custom Germline Variants Workflow Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Custom Germline Variants Workflow Type</td>
<td style="text-align: left;">Specify the name of a custom alignment workflow</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Somatic Variants Workflow URL</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Somatic Variants Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Somatic Variants Workflow Type</td>
<td style="text-align: left;">Other Somatic Variants Workflow Type</td>
<td style="text-align: left;">Custom Somatic Variants Workflow Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Custom Somatic Variants Workflow Type</td>
<td style="text-align: left;">Specify the name of a custom workflow name</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Somatic Variants Sample Type</td>
<td style="text-align: left;">Is the sample case or control in somatic variant analysis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Structural Variant Workflow URL</td>
<td style="text-align: left;">Link to workflow document. DockStore.org recommended. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Structural Variant Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Structural Variant Workflow Type</td>
<td style="text-align: left;">Other Structural Variant Workflow Type</td>
<td style="text-align: left;">Custom Structural Variant Workflow Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Custom Structural Variant Workflow Type</td>
<td style="text-align: left;">Specify the name of a custom workflow name</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Alignment Workflow Url</td>
<td style="text-align: left;">Link to workflow used for read alignment. DockStore.org recommended. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Alignment Workflow Type</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Other Alignment Workflow</td>
<td style="text-align: left;">Other Alignment Workflow</td>
<td style="text-align: left;">Custom Alignment Workflow</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Custom Alignment Workflow</td>
<td style="text-align: left;">Specify the name of a custom alignment workflow</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MSI Workflow Link</td>
<td style="text-align: left;">Link to method workflow (or command) used in estimating the MSI. URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MSI Score</td>
<td style="text-align: left;">Numeric score denoting the aligned reads file's MSI score from MSIsensor. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MSI Status</td>
<td style="text-align: left;">MSIsensor determination of either microsatellite stability or instability.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Genomic Reference</td>
<td style="text-align: left;">Exact version of the human genome reference used in the alignment of reads (e.g. GCF_000001405.39)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Genomic Reference URL</td>
<td style="text-align: left;">Link to human genome sequence (e.g. ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_34/GRCh38.primary_assembly.genome.fa.gz)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Genome Annotation URL</td>
<td style="text-align: left;">Link to the human genome annotation (GTF) file (e.g. ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_34/gencode.v34.annotation.gtf.gz)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Index File Name</td>
<td style="text-align: left;">The name (or part of a name) of a file (of any type). String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Average Base Quality</td>
<td style="text-align: left;">Average base quality collected from samtools. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Average Insert Size</td>
<td style="text-align: left;">Average insert size collected from samtools. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Average Read Length</td>
<td style="text-align: left;">Average read length collected from samtools. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Contamination</td>
<td style="text-align: left;">Fraction of reads coming from cross-sample contamination collected from GATK4. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Contamination Error</td>
<td style="text-align: left;">Estimation error of cross-sample contamination collected from GATK4. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mean Coverage</td>
<td style="text-align: left;">Mean coverage for whole genome sequencing, or mean target coverage for whole exome and targeted sequencing, collected from Picard. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pairs On Diff CHR</td>
<td style="text-align: left;">Pairs on different chromosomes collected from samtools. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Uniquely Mapped</td>
<td style="text-align: left;">Number of reads that map to genome. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Unmapped reads</td>
<td style="text-align: left;">Number of reads that did not map to genome. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Reads Duplicated</td>
<td style="text-align: left;">Proportion of duplicated reads collected from samtools. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Reads Mapped</td>
<td style="text-align: left;">Proportion of mapped reads collected from samtools. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Targets No Coverage</td>
<td style="text-align: left;">Proportion of targets that did not reach 1X coverage over any base from Picard Tools. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Base Mismatch</td>
<td style="text-align: left;">Proportion of mismatched bases collected from samtools. Number</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Coverage 10x</td>
<td style="text-align: left;">Proportion of all reference bases for whole genome sequencing, or targeted bases for whole exome and targeted sequencing, that achieves 10X or greater coverage from Picard Tools.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Mitochondrial Reads</td>
<td style="text-align: left;">Proportion of reads mapping to mitochondria.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Proportion Coverage 30X</td>
<td style="text-align: left;">Proportion of all reference bases for whole genome sequencing, or targeted bases for whole exome and targeted sequencing, that achieves 30X or greater coverage from Picard Tools.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Short Reads</td>
<td style="text-align: left;">Number of reads that were too short. Integer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pseudo Alignment Used</td>
<td style="text-align: left;">Pseudo aligners such as Kallisto or Salmon do not produce aligned reads BAM files. True indicates pseudoalignment was used.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Software and Version</td>
<td style="text-align: left;">Name of software used to generate expression values. String</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Pseudo Alignment Used</td>
<td style="text-align: left;">Pseudo aligner was used</td>
<td style="text-align: left;">Workflow Link, Software and Version, Genomic Reference, Genomic Reference URL</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Data Category</td>
<td style="text-align: left;">Specific content type of the data file.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Expression Units</td>
<td style="text-align: left;">How quantities are corrected for gene length</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Fusion Gene Detected</td>
<td style="text-align: left;">Was a fusion gene identified?</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Fusion Gene Detected</td>
<td style="text-align: left;">A fusion gene was detected</td>
<td style="text-align: left;">Fusion Gene Identity</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fusion Gene Identity</td>
<td style="text-align: left;">The gene symbols of fused genes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Fusion Gene</td>
<td style="text-align: left;">Other fusion gene detected.</td>
<td style="text-align: left;">Specify Other Fusion Gene</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Specify Other Fusion Gene</td>
<td style="text-align: left;">Specify fusion gene detected, if not in list</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Matrix Type</td>
<td style="text-align: left;">Type of data stored in matrix.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Linked Matrices</td>
<td style="text-align: left;">All matrices associated with every part of a SingleCellExperiment object. Comma-delimited list of filenames</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biospecimen Type</td>
<td style="text-align: left;">Biospecimen Type</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Analyte Biospecimen Type</td>
<td style="text-align: left;">A molecular derivative (I.e. RNA / DNA / Protein Lysate) obtained from a specimen</td>
<td style="text-align: left;">Analyte Type, Fixation Duration, Slide Charge Type, Section Thickness Value, Sectioning Days from Index, Shipping Condition Type, Ischemic Time, Ischemic Temperature</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tissue Biospecimen Type</td>
<td style="text-align: left;">Tissue biospecimen</td>
<td style="text-align: left;">Ischemic Time, Ischemic Temperature, Site of Resection or Biopsy, Specimen Laterality, Portion Weight, Total Volume, Tumor Tissue Type, Histologic Morphology Code, Preservation Method, Biospecimen Dimension 1, Biospecimen Dimension 2, Biospecimen Dimension 3, Section Number in Sequence</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bone Marrow Biospecimen Type</td>
<td style="text-align: left;">Bone Marrow biospecimen</td>
<td style="text-align: left;">Ischemic Time, Ischemic Temperature, Site of Resection or Biopsy, Specimen Laterality, Portion Weight, Total Volume, Tumor Tissue Type, Histologic Morphology Code, Preservation Method, Biospecimen Dimension 1, Biospecimen Dimension 2, Biospecimen Dimension 3, Section Number in Sequence</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Urine Biospecimen Type</td>
<td style="text-align: left;">Urine biospecimen</td>
<td style="text-align: left;">Ischemic Time, Ischemic Temperature, Site of Resection or Biopsy, Specimen Laterality, Portion Weight, Total Volume, Tumor Tissue Type, Histologic Morphology Code, Preservation Method</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Blood Biospecimen Type</td>
<td style="text-align: left;">Blood biospecimen</td>
<td style="text-align: left;">Shipping Condition Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Timepoint Label</td>
<td style="text-align: left;">Label to identify the time point at which the clinical data or biospecimen was obtained (e.g. Baseline, End of Treatment, Overall survival, Final). NO PHI/PII INFORMATION IS ALLOWED.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Collection Days from Index</td>
<td style="text-align: left;">Number of days from the research participant's index date that the biospecimen was obtained. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Protocol Link</td>
<td style="text-align: left;">Protocols.io ID or DOI link to a free/open protocol resource describing in detail the assay protocol (e.g. surface markers used in Smart-seq, dissociation duration,  lot/batch numbers for key reagents such as primers, sequencing reagent kits, etc.) or the protocol by which the sample was obtained or generated.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Adjacent Biospecimen IDs</td>
<td style="text-align: left;">List of HTAN Identifiers (separated by commas) of adjacent biospecimens cut from the same sample; for example HTA3_3000_3, HTA3_3000_4, ...</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mounting Medium</td>
<td style="text-align: left;">The solution in which the specimen is embedded, generally under a cover glass. It may be liquid, gum or resinous, soluble in water, alcohol or other solvents and be sealed from the external atmosphere by non-soluble ringing media</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Analyte Type</td>
<td style="text-align: left;">The kind of molecular specimen analyte: a molecular derivative (I.e. RNA / DNA / Protein Lysate) obtained from a specimen</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Acquisition Method Type</td>
<td style="text-align: left;">Records the method of acquisition or source for the specimen under consideration.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Other Acquisition Method</td>
<td style="text-align: left;">A custom acquisition method</td>
<td style="text-align: left;">Acquisition Method Other Specify</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Acquisition Method Other Specify</td>
<td style="text-align: left;">A custom acquisition method [Text - max length 100 characters]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Preservation Method</td>
<td style="text-align: left;">Text term that represents the method used to preserve the sample.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Fixative Type</td>
<td style="text-align: left;">Text term to identify the type of fixative used to preserve a tissue specimen</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Fixation Duration</td>
<td style="text-align: left;">The length of time, from beginning to end, required to process or preserve biospecimens in fixative (measured in minutes)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Ischemic Time</td>
<td style="text-align: left;">Duration of time, in seconds, between when the specimen stopped receiving oxygen and when it was preserved or processed. Integer value.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ischemic Temperature</td>
<td style="text-align: left;">Specify whether specimen experienced warm or cold ischemia.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Collection Media</td>
<td style="text-align: left;">Material Specimen is collected into post procedure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Specimen Laterality</td>
<td style="text-align: left;">For tumors in paired organs, designates the side on which the specimen was obtained.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Portion Weight</td>
<td style="text-align: left;">Numeric value that represents the sample portion weight, measured in milligrams.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Volume</td>
<td style="text-align: left;">Numeric value for the total amount of sample or specimen</td>
<td style="text-align: left;">Total Volume Unit</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Volume Unit</td>
<td style="text-align: left;">Unit of measurement used for the total amount of sample or specimen</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Tissue Type</td>
<td style="text-align: left;">Text that describes the kind of disease present in the tumor specimen as related to a specific timepoint (add rows to select multiple values along with timepoints)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Histologic Morphology Code</td>
<td style="text-align: left;">The microscopic anatomy of normal and abnormal cells and tissues of the specimen as captured in the morphology codes of the International Classification of Diseases for Oncology, 3rd Edition (ICD-O-3). Example - 8010/0</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Preinvasive Morphology</td>
<td style="text-align: left;">Histologic Morphology not included in ICD-O-3 morphology codes, for preinvasive lesions included in the HTAN</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slide Charge Type</td>
<td style="text-align: left;">A description of the charge on the glass slide.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Section Thickness Value</td>
<td style="text-align: left;">Numeric value to describe the thickness of a slice to tissue taken from a biospecimen, measured in microns (um).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Sectioning Days from Index</td>
<td style="text-align: left;">Number of days from the research participant's index date that the biospecimen was sectioned after collection. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Storage Method</td>
<td style="text-align: left;">The method by which a biomaterial was stored after preservation or before another protocol was used.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Processing Days from Index</td>
<td style="text-align: left;">Number of days from the research participant's index date that the biospecimen was processed. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Shipping Condition Type</td>
<td style="text-align: left;">Text descriptor of the shipping environment of a biospecimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Site Data Source</td>
<td style="text-align: left;">Text to identify the data source for the specimen/sample from within the HTAN center, if applicable. Any identifier used within the center to identify data sources. No PHI/PII is allowed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Processing Location</td>
<td style="text-align: left;">Site with an HTAN center where specimen processing occurs, if applicable. Any identifier used within the center to identify processing location. No PHI/PII is allowed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Histology Assessment By</td>
<td style="text-align: left;">Text term describing who (in what role) made the histological assessments of the sample</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Histology Assessment Medium</td>
<td style="text-align: left;">The method of assessment used to characterize histology</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Infiltrating Lymphocytes</td>
<td style="text-align: left;">Measure of Tumor-Infiltrating Lymphocytes [Number]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Degree of Dysplasia</td>
<td style="text-align: left;">Information related to the presence of cells that look abnormal under a microscope but are not cancer. Records the degree of dysplasia for the cyst or lesion under consideration.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Dysplasia Fraction</td>
<td style="text-align: left;">Resulting value to represent the number of pieces of dysplasia divided by the total number of pieces. [Text: max length 5]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Number Proliferating Cells</td>
<td style="text-align: left;">Numeric value that represents the count of proliferating cells determined during pathologic review of the sample slide(s).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Eosinophil Infiltration</td>
<td style="text-align: left;">Numeric value to represent the percentage of infiltration by eosinophils in a tumor sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Granulocyte Infiltration</td>
<td style="text-align: left;">Numeric value to represent the percentage of infiltration by granulocytes in a tumor sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Inflam Infiltration</td>
<td style="text-align: left;">Numeric value to represent local response to cellular injury, marked by capillary dilatation, edema and leukocyte infiltration; clinically, inflammation is manifest by redness, heat, pain, swelling and loss of function, with the need to heal damaged tissue.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Lymphocyte Infiltration</td>
<td style="text-align: left;">Numeric value to represent the percentage of infiltration by lymphocytes in a solid tissue normal sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Monocyte Infiltration</td>
<td style="text-align: left;">Numeric value to represent the percentage of monocyte infiltration in a sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Necrosis</td>
<td style="text-align: left;">Numeric value to represent the percentage of cell death in a malignant tumor sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Neutrophil Infiltration</td>
<td style="text-align: left;">Numeric value to represent the percentage of infiltration by neutrophils in a tumor sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Normal Cells</td>
<td style="text-align: left;">Numeric value to represent the percentage of normal cell content in a malignant tumor sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Stromal Cells</td>
<td style="text-align: left;">Numeric value to represent the percentage of reactive cells that are present in a malignant tumor sample or specimen but are not malignant such as fibroblasts, vascular structures, etc.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Tumor Cells</td>
<td style="text-align: left;">Numeric value that represents the percentage of infiltration by tumor cells in a sample.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Tumor Nuclei</td>
<td style="text-align: left;">Numeric value to represent the percentage of tumor nuclei in a malignant neoplasm sample or specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fiducial Marker</td>
<td style="text-align: left;">Imaging specific: fiducial markers for the alignment of images taken across multiple rounds of imaging.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Slicing Method</td>
<td style="text-align: left;">Imaging specific: the method by which the tissue was sliced.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lysis Buffer</td>
<td style="text-align: left;">scRNA-seq specific: Type of lysis buffer used</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Method of Nucleic Acid Isolation</td>
<td style="text-align: left;">Bulk RNA &amp; DNA-seq specific: method used for nucleic acid isolation. E.g. Qiagen Allprep, Qiagen miRNAeasy. [Text - max length 100]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biospecimen Dimension 1</td>
<td style="text-align: left;">First dimension of tissue fragment (number, up to one decimal place) measured in units as defined by the "dimensions_unit" CDE</td>
<td style="text-align: left;">Dimensions Unit</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biospecimen Dimension 2</td>
<td style="text-align: left;">Second dimension of tissue fragment (number, up to one decimal place) measured in units as defined by the "dimensions_unit" CDE</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Biospecimen Dimension 3</td>
<td style="text-align: left;">Third dimension of tissue fragment (number, up to one decimal place) measured in units as defined by the "dimensions_unit" CDE</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Dimensions Unit</td>
<td style="text-align: left;">Unit of measurement used for dimension CDEs in metric system (i.e. cm, mm, etc)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Section Number in Sequence</td>
<td style="text-align: left;">Numeric value (integer, including ranges) provided to a sample in a series of sections (list all adjacent sections in the Adjacent Biospecimen IDs field)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Start Days from Index</td>
<td style="text-align: left;">Number of days from the date of birth (index date) to the date of an event (e.g. exposure to environmental factor, treatment start, etc.). If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Stop Days from Index</td>
<td style="text-align: left;">Number of days from the date of birth (index date) to the end date of the event (e.g. exposure to environmental factor, treatment start, etc.). Note: if the event occurs at a single time point, e.g. a diagnosis or a lab test, the values for this column is 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ethnicity</td>
<td style="text-align: left;">An individual's self-described social and cultural grouping, specifically whether an individual describes themselves as Hispanic or Latino. The provided values are based on the categories defined by the U.S. Office of Management and Business and used by the U.S. Census Bureau.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Gender</td>
<td style="text-align: left;">Text designations that identify gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles. [Identification of gender is based upon self-report and may come from a form, questionnaire, interview, etc.]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Race</td>
<td style="text-align: left;">An arbitrary classification of a taxonomic group that is a division of a species. It usually arises as a consequence of geographical isolation withina a species and is characterized by shared heredity, physical attributes and behavior, and in the case of humans, by common history, nationality, or geographic distribution.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Vital Status</td>
<td style="text-align: left;">The survival state of the person registered on the protocol.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Dead</td>
<td style="text-align: left;">This indicates the participant is dead and defines further required metadata</td>
<td style="text-align: left;">Year of Death, Cause of Death, Cause of Death Source, Days to Death</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Birth</td>
<td style="text-align: left;">Number of days between the date used for index and the date from a person's date of birth represented as a calculated negative number of days. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Year of Death</td>
<td style="text-align: left;">Numeric value to represent the year of the death of an individual.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Country of Residence</td>
<td style="text-align: left;">Country of Residence at enrollment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Age Is Obfuscated</td>
<td style="text-align: left;">The age of the patient has been modified for compliance reasons. The actual age differs from what is reported. Other date intervals for this patient may also be modified.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Year Of Birth</td>
<td style="text-align: left;">Numeric value to represent the calendar year in which an individual was born.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cause of Death</td>
<td style="text-align: left;">The cause of death</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Cause of Death Source</td>
<td style="text-align: left;">The text term used to describe the source used to determine the patient's cause of death.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Death</td>
<td style="text-align: left;">Number of days between the date used for index and the date from a person's date of death represented as a calculated number of days. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Occupation Duration Years</td>
<td style="text-align: left;">The number of years a patient worked in a specific occupation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Premature At Birth</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the patient was premature (less than 37 weeks gestation) at birth.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Weeks Gestation at Birth</td>
<td style="text-align: left;">Numeric value used to describe the number of weeks starting from the approximate date of the biological mother's last menstrual period and ending with the birth of the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Education Level</td>
<td style="text-align: left;">Highest level of education that the patient completed (direct patient-derived information)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Country of Birth</td>
<td style="text-align: left;">Country where the patient was born.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medically Underserved Area</td>
<td style="text-align: left;">Areas or populations designated by HRSA as having too few primary care providers, high infant mortality, high poverty or a high elderly population:  Use patient zip code to find the county the patient lives in by going to https://www.unitedstateszipcodes.org/ - enter the zip code in the main text field and use the associated county on the right side of the result field. Go to data.hrsa.gov website and select "Query Data". Pick the Medically Underserved Areas/Populations (MUA/P) data source in the step 1 menu and select "View Data". Enter the name of the county (<strong><em>_</em></strong> county) in the first "Service Area" column, adding the state in the 5th column may help direct you to the data.  If the designation type in the third column is "medically underserved area" enter "Yes" as the value. If the county generates a "No data available in table" enter "No" as the value.  A value of "Unknown" indicates that sufficient data was not available to look up the value.  If value is yes, complete the Medically_underserved_score data element.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medically Underserved Area - Yes</td>
<td style="text-align: left;">Patient's zip code is in a medically underserved area</td>
<td style="text-align: left;">Medically Underserved Score</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medically Underserved Area - No</td>
<td style="text-align: left;">Patient's zip code is not in a medically underserved area</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medically Underserved Area - Unknown</td>
<td style="text-align: left;">Insufficient data to look up the Medically Underserved Area value</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medically Underserved Score</td>
<td style="text-align: left;">Index of Medical Underservice (IMU) score, a number between 0 and 100, where 0 represents completely underserved and 100 represents best served or least underserved. Use patient zip code to find the county the patient lives in by going to https://www.unitedstateszipcodes.org/. Enter the zip code in the main text field and use the asociated county on the right side of the result field. Go to data.hrsa.gov website and select Query Data. Pick the Medically Underserved Areas/Populations (MUA/P) data source in the step 1 menu and select View Data. Enter the name of the county (______ county) in the first "Service Area" column, adding the state in the 5th column may help direct you to the data. Enter the Index of Medical Underservice Score in the fourth column to one decimal place as the value.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Rural vs Urban</td>
<td style="text-align: left;">Density of population in the county of residence, based on census data (updated last on 4/28/20). Use patient zip code to find the county the patient lives in by going to https://www.unitedstateszipcodes.org/. Enter the zip code in the main text field and use the associated county on the right side of the result field. Go to https://www2.census.gov/programs-surveys/acs/data/covid_19/Data_Profiles_for_HHS/050-County_By_State/. Select the dp02_XX.csv file where XX = the two letter abbreviation for the appropriate state. On row 166 find the total population for the appropriate county.  If the total population is &lt;2,500 enter a value of "Rural Population"; if 2,500 - 50,000 enter a value of "Urban Cluster"; or if &gt;50,000 enter "Urban Population"</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cancer Incidence</td>
<td style="text-align: left;">Incidence of specific cancer type in a defined area (a number between 0 and 100). The rate of incident cases per population of 100,000 persons of a specific type of cancer as designated in the "primary_diagnosis" data element in the county where the patient resides, using the most recent 2013-2017 NCI Cancer Atlas derived data.  Use patient zip code to find the county the patient lives in by going to https://www.unitedstateszipcodes.org/. Enter the zip code in the main text field and use the asociated county on the right side of the result field. On the https://gis.cancer.gov/canceratlas/tableview/ website, choose "Incidence" from the Topic dropdown menu, state of interest from the Area menu, "All Races" from the Race menu, and the cancer type ("Both Sexes" when possible) from the Statistic menu. Find the county of interest and enter the numeric Age-Adjusted Rate per 100,000 as the value.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cancer Incidence Location</td>
<td style="text-align: left;">The county and state in which the patient lives and to which the cancer_incidence data correlates. Record as "County, State" as they appear in the incidence box from which the cancer_incidence data is obtained in the https://gis.cancer.gov/canceratlas/tableview/ website</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relationship Gender</td>
<td style="text-align: left;">The text term used to describe the gender of the patient's relative with a history of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relationship Age at Diagnosis</td>
<td style="text-align: left;">The age (in years) when the patient's relative was first diagnosed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relationship Primary Diagnosis</td>
<td style="text-align: left;">The text term used to describe the malignant diagnosis of the patient's relative with a history of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relationship Type</td>
<td style="text-align: left;">The subgroup that describes the state of connectedness between members of the unit of society organized around kinship ties.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relative with Cancer History</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether any of the patient's relatives have a history of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Relatives with Cancer History Count</td>
<td style="text-align: left;">The number of relatives the patient has with a known history of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Cancer History Relative</td>
<td style="text-align: left;">Individual has a relative with cancer history</td>
<td style="text-align: left;">Relatives with Cancer History Count, Relationship Type, Relationship Primary Diagnosis, Relationship Gender,Relationship Age at Diagnosis</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Smoking Exposure</td>
<td style="text-align: left;">Indicate if individual has smoking exposure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Smoking Exposure</td>
<td style="text-align: left;">Individual has been exposed to smoke; requires additional metadata</td>
<td style="text-align: left;">Years Smoked, Pack Years Smoked, Cigarettes per Day, Smoking Frequency, Type of Smoke Exposure, Time between Waking and First Smoke, Tobacco Smoking Onset Year, Tobacco Smoking Quit Year, Tobacco Smoking Status, Type of Tobacco Used, Secondhand Smoke as Child, Smoke Exposure Duration, Tobacco Use per Day, Smokeless Tobacco Quit Age</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pack Years Smoked</td>
<td style="text-align: left;">Numeric computed value to represent lifetime tobacco exposure defined as number of cigarettes smoked per day x number of years smoked divided by 20.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Years Smoked</td>
<td style="text-align: left;">Numeric value (or unknown) to represent the number of years a person has been smoking.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol Exposure</td>
<td style="text-align: left;">Indicate if individual has alcohol exposure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Alcohol Exposure</td>
<td style="text-align: left;">Individual has been exposed to alcohol</td>
<td style="text-align: left;">Alcohol Days Per Week, Alcohol Drinks Per Day, Alcohol History, Alcohol Intensity, Alcohol Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol Days Per Week</td>
<td style="text-align: left;">Numeric value used to describe the average number of days each week that a person consumes an alcoholic beverage.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol Drinks Per Day</td>
<td style="text-align: left;">Numeric value used to describe the average number of alcoholic beverages a person consumes per day.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol History</td>
<td style="text-align: left;">A response to a question that asks whether the participant has consumed at least 12 drinks of any kind of alcoholic beverage in their lifetime.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol Intensity</td>
<td style="text-align: left;">Category to describe the patient's current level of alcohol use as self-reported by the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alcohol Type</td>
<td style="text-align: left;">Type of alcohol use</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Asbestos Exposure</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the patient was exposed to asbestos.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cigarettes per Day</td>
<td style="text-align: left;">The average number of cigarettes smoked per day.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Coal Dust Exposure</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether a patient was exposed to fine powder derived by the crushing of coal.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Environmental Tobacco Smoke Exposure</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether a patient was exposed to smoke that is emitted from burning tobacco, including cigarettes, pipes, and cigars. This includes tobacco smoke exhaled by smokers.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Radon Exposure</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the patient was exposed to radon.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Respirable Crystalline Silica Exposure</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether a patient was exposured to respirable crystalline silica, a widespread, naturally occurring, crystalline metal oxide that consists of different forms including quartz, cristobalite, tridymite, tripoli, ganister, chert and novaculite.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Smoking Frequency</td>
<td style="text-align: left;">The text term used to generally decribe how often the patient smokes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Secondhand Smoke as Child</td>
<td style="text-align: left;">The text term used to indicate whether the patient was exposed to secondhand smoke as a child.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Smoke Exposure Duration</td>
<td style="text-align: left;">Text term used to describe the length of time the patient was exposed to an environmental factor.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Type of Smoke Exposure</td>
<td style="text-align: left;">The text term used to describe the patient's specific type of smoke exposure.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Marijuana smoke</td>
<td style="text-align: left;">Marijuana smoke exposure</td>
<td style="text-align: left;">Marijuana Use Per Week</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Marijuana Use Per Week</td>
<td style="text-align: left;">Numeric value that represents the number of times the patient uses marijuana each day.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tobacco Use per Day</td>
<td style="text-align: left;">Numeric value that represents the number of times the patient uses tobacco each day.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Smokeless Tobacco Quit Age</td>
<td style="text-align: left;">Smokeless tobacco quit age</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Time between Waking and First Smoke</td>
<td style="text-align: left;">The text term used to describe the approximate amount of time elapsed between the time the patient wakes up in the morning to the time they smoke their first cigarette.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tobacco Smoking Onset Year</td>
<td style="text-align: left;">The year in which the participant began smoking.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tobacco Smoking Quit Year</td>
<td style="text-align: left;">The year in which the participant quit smoking.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tobacco Smoking Status</td>
<td style="text-align: left;">Category describing current smoking status and smoking history as self-reported by a patient</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Type of Tobacco Used</td>
<td style="text-align: left;">The text term used to describe the specific type of tobacco used by the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Follow Up</td>
<td style="text-align: left;">Number of days between the date used for index and the date of the patient's last follow-up appointment or contact. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Adverse Event</td>
<td style="text-align: left;">Text that represents the Common Terminology Criteria for Adverse Events low level term name for an adverse event.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">BMI</td>
<td style="text-align: left;">A calculated numerical quantity that represents an individual's weight to height ratio.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cause of Response</td>
<td style="text-align: left;">The text term used to describe the suspected cause or reason for the patient disease response.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Comorbidity</td>
<td style="text-align: left;">The text term used to describe a comorbidity disease, which coexists with the patient's malignant disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Comorbidity Method of Diagnosis</td>
<td style="text-align: left;">The text term used to describe the method used to diagnose the patient's comorbidity disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Adverse Event</td>
<td style="text-align: left;">Number of days between the date used for index and the date of the patient's adverse event. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Comorbidity</td>
<td style="text-align: left;">Number of days between the date used for index and the date the patient was diagnosed with a comorbidity. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Progression</td>
<td style="text-align: left;">Number of days between the date used for index and the date the patient's disease progressed. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Progression Free</td>
<td style="text-align: left;">Number of days between the date used for index and the date the patient's disease was formally confirmed as progression-free. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Recurrence</td>
<td style="text-align: left;">Number of days between the date used for index and the date the patient's disease recurred. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Diabetes Treatment Type</td>
<td style="text-align: left;">Text term used to describe the types of treatment used to manage diabetes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Disease Response</td>
<td style="text-align: left;">Code assigned to describe the patient's response or outcome to the disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">DLCO Ref Predictive Percent</td>
<td style="text-align: left;">The value, as a percentage of predicted lung volume, measuring the amount of carbon monoxide detected in a patient's lungs.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">ECOG Performance Status</td>
<td style="text-align: left;">The ECOG functional performance status of the patient/participant.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FEV1 FVC Post Bronch Percent</td>
<td style="text-align: left;">Percentage value to represent result of Forced Expiratory Volume in 1 second (FEV1) divided by the Forced Vital Capacity (FVC) post-bronchodilator.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FEV 1 FVC Pre Bronch Percent</td>
<td style="text-align: left;">Percentage value to represent result of Forced Expiratory Volume in 1 second (FEV1) divided by the Forced Vital Capacity (FVC) pre-bronchodilator.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FEV1 Ref Post Bronch Percent</td>
<td style="text-align: left;">The percentage comparison to a normal value reference range of the volume of air that a patient can forcibly exhale from the lungs in one second post-bronchodilator.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">FEV1 Ref Pre Bronch Percent</td>
<td style="text-align: left;">The percentage comparison to a normal value reference range of the volume of air that a patient can forcibly exhale from the lungs in one second pre-bronchodilator.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Height</td>
<td style="text-align: left;">The height of the patient in centimeters.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hepatitis Sustained Virological Response</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the patient received treatment for a risk factor the patient had at the time of or prior to their diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HPV Positive Type</td>
<td style="text-align: left;">Text classification to represent the strain or type of human papillomavirus identified in an individual.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Karnofsky Performance Status</td>
<td style="text-align: left;">Text term used to describe the classification used of the functional capabilities of a person.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Menopause Status</td>
<td style="text-align: left;">Text term used to describe the patient's menopause status.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Adverse Event Grade</td>
<td style="text-align: left;">The text term used to describe a specific histone variants, which are proteins that substitute for the core canonical histones.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AIDS Risk Factors</td>
<td style="text-align: left;">The text term used to describe a risk factor of the acquired immunodeficiency syndrome (AIDS) that the patient either had at time time of the study or experienced in the past.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Body Surface Area</td>
<td style="text-align: left;">Numeric value used to represent the 2-dimensional extent of the body surface relating height to weight.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CD4 Count</td>
<td style="text-align: left;">The text term used to describe the outcome of the procedure to determine the amount of the CD4 expressing cells in a sample.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CDC HIV Risk Factors</td>
<td style="text-align: left;">The text term used to describe a risk factor for human immunodeficiency virus, as described by the Center for Disease Control.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Imaging</td>
<td style="text-align: left;">Number of days between the date used for index and the date the imaging or scan was performed on the patient. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Evidence of Recurrence Type</td>
<td style="text-align: left;">The text term used to describe the type of evidence used to determine whether the patient's disease recurred</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HAART Treatment Indicator</td>
<td style="text-align: left;">The text term used to indicate whether the patient received Highly Active Antiretroviral Therapy (HAART).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HIV Viral Load</td>
<td style="text-align: left;">Numeric value that represents the concentration of an analyte or aliquot extracted from the sample or sample portion, measured in milligrams per milliliter.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hormonal Contraceptive Use</td>
<td style="text-align: left;">The text term used to indicate whether the patient used hormonal contraceptives.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hysterectomy Margins Involved</td>
<td style="text-align: left;">The text term used to indicate whether the patient's disease was determined to be involved based on the surgical margins of the hysterectomy.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hysterectomy Type</td>
<td style="text-align: left;">The text term used to describe the type of hysterectomy the patient had.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Result</td>
<td style="text-align: left;">The text term used to describe the result of the imaging or scan performed on the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Imaging Type</td>
<td style="text-align: left;">The text term used to describe the type of imaging or scan performed on the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immunosuppressive Treatment Type</td>
<td style="text-align: left;">The text term used to describe the type of immunosuppresive treatment the patient received.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Nadir CD4 Count</td>
<td style="text-align: left;">Numeric value that represents the lowest point to which the CD4 count has dropped (nadir).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pregnancy Outcome</td>
<td style="text-align: left;">The text term used to describe the type of pregnancy the patient had</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Recist Targeted Regions Number</td>
<td style="text-align: left;">Numeric value that represents the number of baseline target lesions, as described by the Response Evaluation Criteria in Solid Tumours (RECIST) criteria</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Recist Targeted Regions Sum</td>
<td style="text-align: left;">Numeric value that represents the sum of baseline target lesions, as described by the Response Evaluation Criteria in Solid Tumours (RECIST) criteria.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Scan Tracer Used</td>
<td style="text-align: left;">The text term used to describe the type of tracer used during the imaging or scan of the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Progression or Recurrence</td>
<td style="text-align: left;">Yes/No/unknown indicator to identify whether a patient has had a new tumor event after initial treatment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Progression or Recurrence</td>
<td style="text-align: left;">The patient has had a new tumor event after initial treatment</td>
<td style="text-align: left;">Progression or Recurrence Type, Days to Progression, Days to Progression Free, Days to Recurrence, Progression or Recurrence Anatomic Site</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Progression or Recurrence Anatomic Site</td>
<td style="text-align: left;">The text term used to describe the anatomic site of resection; biopsy; tissue or organ of biospecimen origin; progression or recurrent disease; treatment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Anatomic Site</td>
<td style="text-align: left;">The text term used to describe the anatomic site of resection; biopsy; tissue or organ of biospecimen origin; progression or recurrent disease; treatment</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NCI Atlas Cancer Site</td>
<td style="text-align: left;">The primary tumor site used to calculate the incidence rate using the NCI Cancer Atlas, a digital atlas which provides geographical data related to cancer utilizing the Surveillance, Epidemiology, and End Results (SEER) Program cancer incidence rates for 2013 to 2017</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Progression or Recurrence Type</td>
<td style="text-align: left;">The text term used to describe the type of progressive or recurrent disease or relapsed disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Reflux Treatment Type</td>
<td style="text-align: left;">Text term used to describe the types of treatment used to manage gastroesophageal reflux disease (GERD).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Risk Factor</td>
<td style="text-align: left;">The text term used to describe a risk factor the patient had at the time of or prior to their diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Risk Factor Treatment</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the patient received treatment for a risk factor the patient had at the time of or prior to their diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Viral Hepatitis Serologies</td>
<td style="text-align: left;">Text term that describes the kind of serological laboratory test used to determine the patient's hepatitus status.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Weight</td>
<td style="text-align: left;">The weight of the patient measured in kilograms.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Treatment End</td>
<td style="text-align: left;">Number of days between the date used for index and the date the treatment ended. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Treatment Start</td>
<td style="text-align: left;">Number of days between the date used for index and the date the treatment started. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Initial Disease Status</td>
<td style="text-align: left;">The text term used to describe the status of the patient's malignancy when the treatment began.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Regimen or Line of Therapy</td>
<td style="text-align: left;">The text term used to describe the regimen or line of therapy.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Therapeutic Agents</td>
<td style="text-align: left;">Text identification of the individual agent(s) used as part of a treatment regimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Effect</td>
<td style="text-align: left;">The text term used to describe the pathologic effect a treatment(s) had on the tumor.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Intent Type</td>
<td style="text-align: left;">Text term to identify the reason for the administration of a treatment regimen. [Manually-curated]</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment or Therapy</td>
<td style="text-align: left;">A yes/no/unknown/not applicable indicator related to the administration of therapeutic agents received.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Outcome</td>
<td style="text-align: left;">Text term that describes the patient's final outcome after the treatment was administered.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Type</td>
<td style="text-align: left;">Text term that describes the kind of treatment administered.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Chemo Concurrent to Radiation</td>
<td style="text-align: left;">The text term used to describe whether the patient was receiving chemotherapy concurrent to radiation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Number of Cycles</td>
<td style="text-align: left;">The numeric value used to describe the number of cycles of a specific treatment or regimen the patient received.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Reason Treatment Ended</td>
<td style="text-align: left;">The text term used to describe the reason a specific treatment or regimen ended.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Arm</td>
<td style="text-align: left;">Text term used to describe the treatment arm assigned to a patient at the time eligibility is determined.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Dose</td>
<td style="text-align: left;">The numeric value used to describe the dose of an agent the patient received.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Dose Units</td>
<td style="text-align: left;">The text term used to describe the dose units of an agent the patient received.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Effect Indicator</td>
<td style="text-align: left;">The text term used to indicate whether the treatment had an effect on the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Treatment Frequency</td>
<td style="text-align: left;">The text term used to describe the frequency the patient received an agent or regimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sentinel Lymph Node Count</td>
<td style="text-align: left;">Numeric count of sentinel lymph nodes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sentinel Node Positive Assessment Count</td>
<td style="text-align: left;">The number or amount of metastatic neoplasms related to the confirmed presence of disease or specific microorganisms during examination of the first rounded mass of lymphatic tissue to which cancer is likely to spread from the primary tumor.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Extranodal Extension Indicator</td>
<td style="text-align: left;">The indicator to determine extranodal involvement or extent of the disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Satellite Metastasis Present Indicator</td>
<td style="text-align: left;">A yes/no indicator to ask if intransit metastases or satellite lesions are present.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Other Biopsy Resection Site</td>
<td style="text-align: left;">A description of the location on or within the human body where the surgical biopsy/resection procedure was performed (Not covered under HTAN Clinical Data Tier 1)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Extent of Tumor Resection</td>
<td style="text-align: left;">The degree to which the lesion has been cut out, or resected.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Precancerous Condition Type</td>
<td style="text-align: left;">The classification of pre-cancerous cells found in a specific collection of data being studied by the Consortium for Molecular and Cellular Characterization of Screen-Detected Lesions (MCL).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prior Sites of Radiation</td>
<td style="text-align: left;">The anatomic location to which radiation treatment was administered to a patient prior to enrollment on a protocol.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immunosuppression</td>
<td style="text-align: left;">The indicator that describes whether or not immunosuppressive therapy was administered.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Concomitant Medication Received Type</td>
<td style="text-align: left;">An enumerated list of the type of concomitant medication received by the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Family Member Vital Status Indicator</td>
<td style="text-align: left;">The response indicates whether the family member of the patient with a history of cancer is alive. (Extension to GDC attributes in Family History Tier 1)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Occurrence Indicator</td>
<td style="text-align: left;">The indicator that describes whether or not a COVID-19 infectious disorder occurred.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Current Status</td>
<td style="text-align: left;">The patient's current COVID-19 status of sign or symptom events or interventions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Positive Lab Test Indicator</td>
<td style="text-align: left;">The indicator that describes whether or not there was a COVID-19 positive test result.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Antibody Testing</td>
<td style="text-align: left;">Text term that demonstrates the test results of immunoglobulin M (IgM) and immunoglobulin G (IgG) antibodies to the SARS-CoV-2 virus in subject serum samples.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Complications Severity</td>
<td style="text-align: left;">Text term that retrospectively indicates the worst complications during COVID-19 infectious disorder in the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">COVID19 Cancer Treatment Followup</td>
<td style="text-align: left;">Indicator that describes if cancer treatment was modified for the patient due to COVID-19 infectious disorder</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ecig vape use</td>
<td style="text-align: left;">Use of non-traditional cigarette nicotine delivery device (electronic cigarette, ENDS - electronic nicotine delivery system)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ecig vape 30 day use num</td>
<td style="text-align: left;">Number of days e-cigarettes or vaping device was used in the last 30 days</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ecig vape times per day</td>
<td style="text-align: left;">e-cig frequency of use (times per day—one “time” consists of around 15 puffs or lasts around 10 minutes)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Type of smoke exposure cumulative years</td>
<td style="text-align: left;">The number of cumulative years of the patient's specific type of smoke exposure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Chewing tobacco daily use count</td>
<td style="text-align: left;">The quantity of daily use of tobacco, in the form of a plug, usually flavored, for chewing rather than smoking.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Second hand smoke exposure years</td>
<td style="text-align: left;">The number of cumulative years of the patient's exposure to second-hand cigarette smoke</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Known Genetic Predisposition Mutation</td>
<td style="text-align: left;">A yes/no/unknown indicator to identify whether there is a known genetic predisposition mutation present in the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hereditary Cancer Predisposition Syndrome</td>
<td style="text-align: left;">History of presence of inherited genetic predisposition syndrome that confers heightened susceptibility to cancer in the patient.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cancer Associated Gene Mutations</td>
<td style="text-align: left;">Type of inherited germline or other gene mutations that confers heightened susceptibility to cancer identified in patient history</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mutational Signatures</td>
<td style="text-align: left;">Mutational signatures identified in the patient, includes signatures linked to selected exogenous carcinogens, endogenous and enzymatic modification of DNA or defective DNA repair. Note: Include only outputs of tests that were completed clinically for the participant and only include data from a diagnostic array that was completed prior to research sequencing was done.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mismatch Repair System Status</td>
<td style="text-align: left;">The text that best describes the condition or state of MMR (mismatch repair system) in the patient</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lab Tests for MMR Status</td>
<td style="text-align: left;">Laboratory tests used to evaluate the status of mismatch repair pathways</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mode of Cancer Detection</td>
<td style="text-align: left;">Text term used to describe the mode of cancer detection, like standard of care screening or random detection</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Gene Symbol</td>
<td style="text-align: left;">The text term used to describe a gene targeted or included in molecular analysis. For rearrangements, this is should be used to represent the reference gene. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Molecular Analysis Method</td>
<td style="text-align: left;">The text term used to describe the method used for molecular analysis. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Test Result</td>
<td style="text-align: left;">The text term used to describe the result of the molecular test. If the test result was a numeric value see test_value. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">AA Change</td>
<td style="text-align: left;">Alphanumeric value used to describe the amino acid change for a specific genetic variant. Example: R116Q. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Antigen</td>
<td style="text-align: left;">The text term used to describe an antigen included in molecular testing. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Clinical Biospecimen Type</td>
<td style="text-align: left;">The text term used to describe the biological material used for testing, diagnostic, treatment or research purposes. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Blood Test Normal Range Upper</td>
<td style="text-align: left;">Numeric value used to describe the upper limit of the normal range used to describe a healthy individual at the institution where the test was completed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Blood Test Normal Range Lower</td>
<td style="text-align: left;">Numeric value used to describe the lower limit of the normal range used to describe a healthy individual at the institution where the test was completed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cell Count</td>
<td style="text-align: left;">Numeric value used to describe the number of cells used for molecular testing. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Chromosome</td>
<td style="text-align: left;">The text term used to describe a chromosome targeted or included in molecular testing. If a specific genetic variant is being reported, this property can be used to capture the chromosome where that variant is located. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Clonality</td>
<td style="text-align: left;">The text term used to describe whether a genomic variant is related by descent from a single progenitor cell. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Copy Number</td>
<td style="text-align: left;">Numeric value used to describe the number of times a section of the genome is repeated or copied within an insertion, duplication or deletion variant. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cytoband</td>
<td style="text-align: left;">Alphanumeric value used to describe the cytoband or chromosomal location targeted or included in molecular analysis. If a specific genetic variant is being reported, this property can be used to capture the cytoband where the variant is located. Format: [chromosome][chromosome arm].[band+sub-bands]. Example: 17p13.1. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Exon</td>
<td style="text-align: left;">Exon number targeted or included in a molecular analysis. If a specific genetic variant is being reported, this property can be used to capture the exon where that variant is located. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Histone Family</td>
<td style="text-align: left;">The text term used to describe the family, or classification of a group of basic proteins found in chromatin, called histones.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Histone Variant</td>
<td style="text-align: left;">The text term used to describe a specific histone variants, which are proteins that substitute for the core canonical histones.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Intron</td>
<td style="text-align: left;">Intron number targeted or included in molecular analysis. If a specific genetic variant is being reported, this property can be used to capture the intron where that variant is located.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Laboratory Test</td>
<td style="text-align: left;">The text term used to describe the medical testing used to diagnose, treat or further understand a patient's disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Loci Abnormal Count</td>
<td style="text-align: left;">Numeric value used to describe the number of loci determined to be abnormal.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Loci Count</td>
<td style="text-align: left;">Numeric value used to describe the number of loci tested.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Locus</td>
<td style="text-align: left;">Alphanumeric value used to describe the locus of a specific genetic variant. Example: NM_001126114.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mismatch Repair Mutation</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the mutation included in molecular testing was known to have an affect on the mismatch repair process. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Molecular Consequence</td>
<td style="text-align: left;">The text term used to describe the molecular consequence of genetic variation. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pathogenicity</td>
<td style="text-align: left;">The text used to describe a variant's level of involvement in the cause of the patient's disease according to the standards outlined by the American College of Medical Genetics and Genomics (ACMG).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ploidy</td>
<td style="text-align: left;">Text term used to describe the number of sets of homologous chromosomes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Second Exon</td>
<td style="text-align: left;">The second exon number involved in molecular variation. If a specific genetic variant is being reported, this property can be used to capture the second exon where that variant is located. This property is typically used for a translocation where two different locations are involved in the variation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Second Gene Symbol</td>
<td style="text-align: left;">The text term used to describe a secondary gene targeted or included in molecular analysis. For rearrangements, this is should represent the location of the variant. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Specialized Molecular Test</td>
<td style="text-align: left;">Text term used to describe a specific test that is not covered in the list of molecular analysis methods. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Test Analyte Type</td>
<td style="text-align: left;">The text term used to describe the type of analyte used for molecular testing. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Test Units</td>
<td style="text-align: left;">The text term used to describe the units of the test value for a molecular test. This property is used in conjunction with test_value. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Test Value</td>
<td style="text-align: left;">The text term or numeric value used to describe a specific result of a molecular test. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Transcript</td>
<td style="text-align: left;">Alphanumeric value used to describe the transcript of a specific genetic variant. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Variant Origin</td>
<td style="text-align: left;">The text term used to describe the biological origin of a specific genetic variant. Note: This node is meant to capture molecular tests that were completed clinically for the participant and only includes data from diagnostic array that was completed prior to research sequencing was done. Do not include data related to research assay outputs here.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Variant Type</td>
<td style="text-align: left;">The text term used to describe the type of genetic variation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Zygosity</td>
<td style="text-align: left;">The text term used to describe the zygosity of a specific genetic variant.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cog Neuroblastoma Risk Group</td>
<td style="text-align: left;">Text term that represents the categorization of patients on the basis of prognostic factors per a system developed by Children's Oncology Group (COG). Risk level is used to assign treatment intensity.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cog Rhabdomyosarcoma Risk Group</td>
<td style="text-align: left;">Text term used to describe the classification of rhabdomyosarcoma, as defined by the Children's Oncology Group (COG).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Gleason Grade Group</td>
<td style="text-align: left;">The text term used to describe the overall grouping of grades defined by the Gleason grading classification, which is used to determine the aggressiveness of prostate cancer. Note that this grade describes the entire prostatectomy specimen and is not specific to the sample used for sequencing.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Gleason Grade Tertiary</td>
<td style="text-align: left;">The text term used to describe the tertiary pattern as described by the Gleason Grading System.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Gleason Patterns Percent</td>
<td style="text-align: left;">Numeric value that represents the percentage of Patterns 4 and 5, which is used when the Gleason score is greater than 7 to predict prognosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Greatest Tumor Dimension</td>
<td style="text-align: left;">Numeric value that represents the measurement of the widest portion of the tumor in centimeters.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">IGCCCG Stage</td>
<td style="text-align: left;">The text term used to describe the International Germ Cell Cancer Collaborative Group (IGCCCG), a grouping used to further classify metastatic testicular tumors.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">INPC Grade</td>
<td style="text-align: left;">Text term used to describe the classification of neuroblastic differentiation within neuroblastoma tumors, as defined by the International Neuroblastoma Pathology Classification (INPC).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">INPC Histologic Group</td>
<td style="text-align: left;">The text term used to describe the classification of neuroblastomas distinguishing between favorable and unfavorable histologic groups. The histologic score, defined by the International Neuroblastoma Pathology Classification (INPC), is based on age, mitosis-karyorrhexis index (MKI), stromal content and degree of tumor cell differentiation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">INRG Stage</td>
<td style="text-align: left;">The text term used to describe the staging classification of neuroblastic tumors, as defined by the International Neuroblastoma Risk Group (INRG).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">INSS Stage</td>
<td style="text-align: left;">Text term used to describe the staging classification of neuroblastic tumors, as defined by the International Neuroblastoma Staging System (INSS).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">International Prognostic Index</td>
<td style="text-align: left;">The text term used to describe the International Prognostic Index, which classifies the prognosis of patients with aggressive non-Hodgkin's lymphoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">IRS Group</td>
<td style="text-align: left;">Text term used to describe the classification of rhabdomyosarcoma tumors, as defined by the Intergroup Rhabdomyosarcoma Study (IRS).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">IRS Stage</td>
<td style="text-align: left;">The text term used to describe the classification of rhabdomyosarcoma tumors, as defined by the Intergroup Rhabdomyosarcoma Study (IRS).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">ISS Stage</td>
<td style="text-align: left;">The multiple myeloma disease stage at diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lymph Node Involved Site</td>
<td style="text-align: left;">The text term used to describe the anatomic site of lymph node involvement.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Margin Distance</td>
<td style="text-align: left;">Numeric value (in centimeters) that represents the distance between the tumor and the surgical margin.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Margins Involved Site</td>
<td style="text-align: left;">The text term used to describe the anatomic sites that were involved in the survival margins.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Medulloblastoma Molecular Classification</td>
<td style="text-align: left;">The text term used to describe the classification of medulloblastoma tumors based on molecular features.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Micropapillary Features</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether micropapillary features were determined to be present.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mitosis Karyorrhexis Index</td>
<td style="text-align: left;">Text term that represents the component of the International Neuroblastoma Pathology Classification (INPC) for mitosis-karyorrhexis index (MKI).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Non Nodal Regional Disease</td>
<td style="text-align: left;">The text term used to describe whether the patient had non-nodal regional disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Non Nodal Tumor Deposits</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe the presence of tumor deposits in the pericolic or perirectal fat or in adjacent mesentery away from the leading edge of the tumor.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Specimen Status</td>
<td style="text-align: left;">The text term used to describe the physical condition of the involved ovary.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Surface Involvement</td>
<td style="text-align: left;">The text term that describes whether the surface tissue (outer boundary) of the ovary shows evidence of involvement or presence of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pregnant at Diagnosis</td>
<td style="text-align: left;">The text term used to indicate whether the patient was pregnant at the time they were diagnosed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Primary Gleason Grade</td>
<td style="text-align: left;">The text term used to describe the primary Gleason score, which describes the pattern of cells making up the largest area of the tumor. The primary and secondary Gleason pattern grades are combined to determine the patient's Gleason grade group, which is used to determine the aggresiveness of prostate cancer. Note that this grade describes the entire prostatectomy specimen and is not specific to the sample used for sequencing.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Secondary Gleason Grade</td>
<td style="text-align: left;">The text term used to describe the secondary Gleason score, which describes the pattern of cells making up the second largest area of the tumor. The primary and secondary Gleason pattern grades are combined to determine the patient's Gleason grade group, which is used to determine the aggresiveness of prostate cancer. Note that this grade describes the entire prostatectomy specimen and is not specific to the sample used for sequencing.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Supratentorial Localization</td>
<td style="text-align: left;">Text term to specify the location of the supratentorial tumor.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Depth</td>
<td style="text-align: left;">Numeric value that represents the depth of tumor invasion, measured in millimeters (mm).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">WHO CNS Grade</td>
<td style="text-align: left;">WHO CNS Grade</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">WHO NTE Grade</td>
<td style="text-align: left;">WHO NTE Grade</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Age at Diagnosis</td>
<td style="text-align: left;">Age at the time of diagnosis expressed in number of days since birth.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Days to Last Follow up</td>
<td style="text-align: left;">Time interval from the date of last follow up to the date of initial pathologic diagnosis, represented as a calculated number of days. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Days to Last Known Disease Status</td>
<td style="text-align: left;">Time interval from the date of last follow up to the date of initial pathologic diagnosis, represented as a calculated number of days. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Last Known Disease Status</td>
<td style="text-align: left;">Text term that describes the last known state or condition of an individual's neoplasm.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Primary Diagnosis</td>
<td style="text-align: left;">Text term used to describe the patient's histologic diagnosis, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Prior Malignancy</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe the patient's history of prior cancer diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prior Treatment</td>
<td style="text-align: left;">A yes/no/unknown/not applicable indicator related to the administration of therapeutic agents received before the body specimen was collected.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Site of Resection or Biopsy</td>
<td style="text-align: left;">The text term used to describe the anatomic site of the resection or biopsy of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tissue or Organ of Origin</td>
<td style="text-align: left;">The text term used to describe the anatomic site of origin, of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Grade</td>
<td style="text-align: left;">Numeric value to express the degree of abnormality of cancer cells, a measure of differentiation and aggressiveness.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Clinical M</td>
<td style="text-align: left;">Extent of the distant metastasis for the cancer based on evidence obtained from clinical assessment parameters determined prior to treatment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Clinical N</td>
<td style="text-align: left;">Extent of the regional lymph node involvement for the cancer based on evidence obtained from clinical assessment parameters determined prior to treatment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Clinical Stage</td>
<td style="text-align: left;">Stage group determined from clinical information on the tumor (T), regional node (N) and metastases (M) and by grouping cases with similar prognosis for cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Clinical T</td>
<td style="text-align: left;">Extent of the primary cancer based on evidence obtained from clinical assessment parameters determined prior to treatment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Pathologic M</td>
<td style="text-align: left;">Code to represent the defined absence or presence of distant spread or metastases (M) to locations via vascular channels or lymphatics beyond the regional lymph nodes, using criteria established by the American Joint Committee on Cancer (AJCC).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Pathologic N</td>
<td style="text-align: left;">The codes that represent the stage of cancer based on the nodes present (N stage) according to criteria based on multiple editions of the AJCC's Cancer Staging Manual.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Pathologic Stage</td>
<td style="text-align: left;">The extent of a cancer, especially whether the disease has spread from the original site to other parts of the body based on AJCC staging criteria.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Pathologic T</td>
<td style="text-align: left;">Code of pathological T (primary tumor) to define the size or contiguous extension of the primary tumor (T), using staging criteria from the American Joint Committee on Cancer (AJCC).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">AJCC Staging System Edition</td>
<td style="text-align: left;">The text term used to describe the version or edition of the American Joint Committee on Cancer Staging Handbooks, a publication by the group formed for the purpose of developing a system of staging for cancer that is acceptable to the American medical profession and is compatible with other accepted classifications.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Anaplasia Present</td>
<td style="text-align: left;">Yes/no/unknown/Not Reported indicator used to describe whether anaplasia was present at the time of diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Anaplasia Present</td>
<td style="text-align: left;">Indicates anaplasia is present</td>
<td style="text-align: left;">Anaplasia Present Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Anaplasia Present Type</td>
<td style="text-align: left;">The text term used to describe the morphologic findings indicating the presence of a malignant cellular infiltrate characterized by the presence of large pleomorphic cells, necrosis, and high mitotic activity in a tissue sample.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Best Overall Response</td>
<td style="text-align: left;">The best improvement achieved throughout the entire course of protocol treatment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breslow Thickness</td>
<td style="text-align: left;">The number that describes the distance, in millimeters, between the upper layer of the epidermis and the deepest point of tumor penetration.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Classification of Tumor</td>
<td style="text-align: left;">Text that describes the kind of disease present in the tumor specimen as related to a specific timepoint.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Diagnosis</td>
<td style="text-align: left;">Number of days between the date used for index and the date the patient was diagnosed with the malignant disease. If not applicable please enter 'Not Applicable'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">First Symptom Prior to Diagnosis</td>
<td style="text-align: left;">Text term used to describe the patient's first symptom experienced prior to diagnosis and thought to be related to the disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Gross Tumor Weight</td>
<td style="text-align: left;">Numeric value used to describe the gross pathologic tumor weight, measured in grams.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Laterality</td>
<td style="text-align: left;">For tumors in paired organs, designates the side on which the cancer originates.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lymph Nodes Positive</td>
<td style="text-align: left;">The number of lymph nodes involved with disease as determined by pathologic examination.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lymph Nodes Tested</td>
<td style="text-align: left;">The number of lymph nodes tested to determine whether lymph nodes were involved with disease as determined by a pathologic examination.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lymphatic Invasion Present</td>
<td style="text-align: left;">A yes/no indicator to ask if small or thin-walled vessel invasion is present, indicating lymphatic involvement</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Metastasis at Diagnosis</td>
<td style="text-align: left;">The text term used to describe the extent of metastatic disease present at diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Metastasis at Diagnosis Site</td>
<td style="text-align: left;">Text term to identify an anatomic site in which metastatic disease involvement is found.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Method of Diagnosis</td>
<td style="text-align: left;">Text term used to describe the method used to confirm the patients malignant diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mitotic Count</td>
<td style="text-align: left;">The number of mitoses identified under the microscope in tumors. The method of counting varies, according to the specific tumor examined. Usually, the mitotic count is determined based on the number of mitoses per high power field (40X) or 10 high power fields.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Percent Tumor Invasion</td>
<td style="text-align: left;">The percentage of tumor cells spread locally in a malignant neoplasm through infiltration or destruction of adjacent tissue.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Peritoneal Fluid Cytological Status</td>
<td style="text-align: left;">The text term used to describe the malignant status of the peritoneal fluid determined by cytologic testing.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Perineural Invasion Present</td>
<td style="text-align: left;">A yes/no indicator to ask if perineural invasion or infiltration of tumor or cancer is present.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Residual Disease</td>
<td style="text-align: left;">Text terms to describe the status of a tissue margin following surgical resection.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Synchronous Malignancy</td>
<td style="text-align: left;">A yes/no/unknown indicator used to describe whether the patient had an additional malignant diagnosis at the same time the tumor used for sequencing was diagnosed. If both tumors were sequenced, both tumors would have synchronous malignancies.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Confined to Organ of Origin</td>
<td style="text-align: left;">The yes/no/unknown indicator used to describe whether the tumor is confined to the organ where it originated and did not spread to a proximal or distant location within the body.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Focality</td>
<td style="text-align: left;">The text term used to describe whether the patient's disease originated in a single location or multiple locations.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tumor Largest Dimension Diameter</td>
<td style="text-align: left;">Numeric value used to describe the maximum diameter or dimension of the primary tumor, measured in centimeters.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Vascular Invasion Present</td>
<td style="text-align: left;">The yes/no indicator to ask if large vessel or venous invasion was detected by surgery or presence in a tumor specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Vascular Invasion Present</td>
<td style="text-align: left;">Indicates venous invasion was detected by surgery or presence in a tumor specimen</td>
<td style="text-align: left;">Vascular Invasion Type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Vascular Invasion Type</td>
<td style="text-align: left;">Text term that represents the type of vascular tumor invasion.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Year of Diagnosis</td>
<td style="text-align: left;">Numeric value to represent the year of an individual's initial pathologic diagnosis of cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Morphology</td>
<td style="text-align: left;">The third edition of the International Classification of Diseases for Oncology, published in 2000 used principally in tumor and cancer registries for coding the site (topography) and the histology (morphology) of neoplasms. The study of the structure of the cells and their arrangement to constitute tissues and, finally, the association among these to form organs. In pathology, the microscopic process of identifying normal and abnormal morphologic characteristics in tissues, by employing various cytochemical and immunocytochemical stains. A system of numbered categories for representation of data.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Topography Code</td>
<td style="text-align: left;">Topography Code, indicating site within the body, based on ICD-O-3.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Additional Topography</td>
<td style="text-align: left;">Topography not included in the ICD-O-3 Topography codes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Cancer Detection Method Type</td>
<td style="text-align: left;">The means, manner of procedure, or systematic course of actions performed in order to discover or identify lung cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Cancer Participant Procedure History</td>
<td style="text-align: left;">Text name of a surgical or operative procedure used in a natural history protocol of a lung cancer participant.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Adjacent Histology Type</td>
<td style="text-align: left;">The type of morphologic characteristics observed by microscope in the tissue next to a benign or malignant tissue growth</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Tumor Location Anatomic Site</td>
<td style="text-align: left;">Anatomic location of the tumor inside the lung</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Tumor Lobe Bronchial Location</td>
<td style="text-align: left;">Anatomic lobe and bronchial location of the tumor inside the lung</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Current Lung Cancer Symptoms</td>
<td style="text-align: left;">Reported lung cancer related symptoms person is currently experiencing</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Topography</td>
<td style="text-align: left;">Lung PCA specific topography (not covered in previous tiers)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Lung Cancer Harboring Genomic Aberrations</td>
<td style="text-align: left;">Genomic aberrations in participants with lung cancer (specific lung cancer associated gene mutations not covered in Tiers 1 and 2)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Cancer Detection Method Type</td>
<td style="text-align: left;">The means, manner of procedure, or systematic course of actions performed in order to discover or identify colorectal cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">History of Prior Colon Polyps</td>
<td style="text-align: left;">Yes/No indicator to describe if the subject had a previous history of colon polyps as noted in the history/physical or previous endoscopic report (s).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Family Colon Cancer History Indicator</td>
<td style="text-align: left;">The indicator to designate if any first degree relative has a history of colorectal cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Family Medical History Colorectal Polyp Diagnosis</td>
<td style="text-align: left;">A yes/no/unknown/not applicable indicator related to family medical history diagnosis of polypoid lesion that arises from the colon or rectum and protrudes into the lumen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immediate Family History Endometrial Cancer</td>
<td style="text-align: left;">Text that describes the age at which the family member was diagnosed with endometrial or uterine cancer in relationship to their 50th birthday.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immediate Family History Ovarian Cancer</td>
<td style="text-align: left;">Text that describes the age at which the family member was diagnosed with ovarian cancer in relationship to their 50th birthday.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Patient Inflammatory Bowel Disease Personal Medica History</td>
<td style="text-align: left;">The indicator for patient's personal medical history of inflammatory bowel disease (chronic, non-specific disorders of unknown etiology, including Crohn disease and ulcerative colitis).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Patient Colonoscopy Performed Indicator</td>
<td style="text-align: left;">The yes/no indicator that records if the subject has undergone a previous colonoscopy.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Cancer Tumor Border Configuration</td>
<td style="text-align: left;">The description of the border configuration of a colorectal tumor at pathologic assessment.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MLH1 Promoter Methylation Status</td>
<td style="text-align: left;">Text term to define the status of promoter methylation for the MLH1 gene. Note: MLH1 gene is commonly associated with hereditary nonpolyposis colorectal cancer. Testing for methylation of the MLH1 promoter can help distinguish sporadic from inherited cancers.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Cancer KRAS Indicator</td>
<td style="text-align: left;">The yes/no/not applicable indicator that describes if patient has diagnosis of colorectal cancer with known KRAS.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colon Polyp Occurence Indicator</td>
<td style="text-align: left;">Yes/No indicator to describe if the subject had a previous history of colon polyps as noted in the history/physical or previous endoscopic report (s).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Family History Colorectal Polyp</td>
<td style="text-align: left;">A yes/no/unknown/not applicable indicator related to family medical history diagnosis of polypoid lesion that arises from the colon or rectum and protrudes into the lumen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Polyp New Indicator</td>
<td style="text-align: left;">A yes/no response to a question that asks whether any new polyps greater or equal to two millimeter were identified.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Polyp Shape</td>
<td style="text-align: left;">Shape of polyp identified in the participant</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Size of Polyp Removed</td>
<td style="text-align: left;">Size of the polyp removed in cm</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Polyp Count</td>
<td style="text-align: left;">The total number of polyps detected</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Polyp Type</td>
<td style="text-align: left;">Type of polyp found in the participant</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Colorectal Polyp Adenoma Type</td>
<td style="text-align: left;">Type of adenoma associated with the polyp</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Detection Method Type</td>
<td style="text-align: left;">The means, manner of procedure, or systematic course of actions performed in order to discover or identify breast cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Histology Category</td>
<td style="text-align: left;">Classification of the type of invasive breast carcinoma diagnosed based on histologic attributes.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Invasive Lobular Breast Carcinoma Histologic Category</td>
<td style="text-align: left;">The histologic subtype for an infiltrating lobular carcinoma of the breast.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Invasive Ductal Breast Carcinoma Histologic Category</td>
<td style="text-align: left;">The histologic subtype for the most common type of invasive breast carcinoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Biopsy Procedure Finding Type</td>
<td style="text-align: left;">Text term to describe the result of the examination of the breast tissue specimen or fluid as related to the presence and nature of disease.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Quadrant Site</td>
<td style="text-align: left;">The breast quadrant or structure from which the breast tissue specimen was removed for microscopic examination.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Cancer Assessment Tests</td>
<td style="text-align: left;">Text term to identify assessment tests done in participants during diagnosis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Cancer Genomic Test Performed</td>
<td style="text-align: left;">Text term that represents the name of the genomic test performed for breast cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mammaprint Risk Group</td>
<td style="text-align: left;">Text term that represents the risk group for breast cancer as determined by assessment of the MammaPrint test.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Oncotype Risk Group</td>
<td style="text-align: left;">Text term that represents the risk group for breast cancer as determined by assessment of the Oncotype recurrence score.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Estrogen Receptor Status</td>
<td style="text-align: left;">Text term to represent the overall result of Estrogen Receptor (ER) testing in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Progesteroner Receptor Status</td>
<td style="text-align: left;">Text term to represent the overall result of Progresterone Receptor (PR) testing in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Cancer Allred Estrogen Receptor Score</td>
<td style="text-align: left;">The numeric Allred score, that is cell staining percentage plus intensity, to determine estrogen receptor status.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prior Invasive Breast Disease</td>
<td style="text-align: left;">Text term to indicate prior invasive breast condition in the participant</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma ER Status Percentage Value</td>
<td style="text-align: left;">A numerical quantity measured or assigned or computed which captures the estrogen receptor level measured in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma PR Status Percentage Value</td>
<td style="text-align: left;">A numerical quantity measured or assigned or computed which captures the progesterone receptor level measured in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HER2 Breast Carcinoma Copy Number Total</td>
<td style="text-align: left;">Result of HER2 Copy Number testing (in a participant with breast cancer), expressed as a range of values.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Centromere 17 Copy Number</td>
<td style="text-align: left;">Result of Centromere 17 testing in a sample or specimen of metastatic breast carcinoma, expressed as a range of values.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma HER2 Centromere17 Copynumber Total</td>
<td style="text-align: left;">Number of Cells Counted for HER2 &amp; Centromere 17 Copy Numbers in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma HER2 Chromosome17 Ratio</td>
<td style="text-align: left;">HER2 chromosome 17 ratio in participants with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma Surgical Procedure Name</td>
<td style="text-align: left;">Text name of a surgical procedure performed for a person with a diagnosis of breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma HER2 Ratio Diagnosis</td>
<td style="text-align: left;">HER2 ratio of the participant at diagnosis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma HER2 Status</td>
<td style="text-align: left;">Text term to signify the result of the medical procedure that involves testing a sample of blood or tissue for HER2 in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Hormone Therapy Breast Cancer Prevention Indicator</td>
<td style="text-align: left;">Did the patient receive hormonal therapy for prevention of breast cancer?</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma ER Staining Intensity</td>
<td style="text-align: left;">Text term to indicate the ER staining intensity on pathology assessment in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Carcinoma PR Staining Intensity</td>
<td style="text-align: left;">Text term to indicate the PR staining intensity on pathology assessment in a participant with breast cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Oncotype Score</td>
<td style="text-align: left;">OncotypeDX recurrence score</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Breast Imaging Performed Type</td>
<td style="text-align: left;">The kind of technology or method performed for screening, diagnosis, surgical procedures or therapy that aids in the visualization of the breast(s).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multifocal Breast Carcinoma Present Indicator</td>
<td style="text-align: left;">A response to indicate if there is breast cancer characterized by the presence of multiple cancerous tumors that originate from the same clone and usually located in the same quadrant of the breast.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Multicentric Breast Carcinoma Present Indicator</td>
<td style="text-align: left;">A response to indicate if there is breast cancer characterized by the presence of multiple cancerous tumors that originate from different clones and usually located in different quadrants of the breast.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">BIRADS Mammography Breast Density Category</td>
<td style="text-align: left;">The category that describes the relative amount of different tissues present in the breast on a mammogram based on the updated 2015 edition of the American College of Radiology's Breast Imaging Reporting and Data System (BI-RADS) reporting guidelines.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CNS Tumor Primary Anatomic Site</td>
<td style="text-align: left;">Primary tumor location in the central nervous system that comprise the tissues of the central nervous system (brain and spinal cord)-not covered in Tiers 1 and 2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Glioma Specific Metastasis Sites</td>
<td style="text-align: left;">Evidence of active brain metastasis including leptomeningeal involvement</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Glioma Specific Radiation Field</td>
<td style="text-align: left;">A description of the location on or within the CNS where radiation was administered in a partcipant with glioma</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Supra Tentorial Ependymoma Molecular Subgroup</td>
<td style="text-align: left;">Text term to identify the molecular subgroup in a supra tentorial ependymoma</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Infra Tentorial Ependymoma Molecular Subgroup</td>
<td style="text-align: left;">Text term to identify the molecular subgroup in a infra tentorial ependymoma</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Neuroblastoma MYCN Gene Amplification Status</td>
<td style="text-align: left;">Neuroblastoma MYCN amplification or over-expression status</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Specimen Blast Count Percentage Value</td>
<td style="text-align: left;">The value, in percent(%) of the medical procedure that involves testing a sample of blood for blast cells, immature (undifferentiated) cells during diagnosis</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">NCI ALL Risk Group</td>
<td style="text-align: left;">The NCI risk group assigned to a patient at initial diagnosis with Acute Lymphoblastic Leukemia.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MRD ALL Diagnostic Sensitivity</td>
<td style="text-align: left;">The assay sensitivity results of a diagnostic assessment of Minimal Residual Disease in patients diagnosed with Acute Lymphoblastic Leukemia.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">CNS Leukemia Status</td>
<td style="text-align: left;">The status of central nervous system leukemia at the time of diagnosis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Cancer Histologic Subtype</td>
<td style="text-align: left;">Text term to describe the histological subtype of ovarian cancer in the participant</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Cancer Surgical Outcome</td>
<td style="text-align: left;">Text term that describes the kind of surgical treatment administered.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Ovarian Cancer Platinum Status</td>
<td style="text-align: left;">Text term to indicate the status of treatment with platinum in participant with ovarian cancer</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Location Extent Extraprostatic Extension</td>
<td style="text-align: left;">Location and extent of extraprostatic extension</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Location Nature Positive Margins</td>
<td style="text-align: left;">Location and nature of positive margins</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Seminal Vesicle Invasion</td>
<td style="text-align: left;">An anatomic position identifying a side of the body where local spread of malignant neoplasm is found to infiltrate tissue in the saclike glandular diverticulum on the ductus deferens in a male.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prostate Carcinoma Histologic Type</td>
<td style="text-align: left;">The diagnostic subclassification of an invasive prostate carcinoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prostate Cancer Local Extent</td>
<td style="text-align: left;">The response used to categorize the local extent of disease for prostate cancer.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Additonal Findings Uninvolved Prostate</td>
<td style="text-align: left;">Additional findings, uninvolved prostate</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Prostate Cancer Cytologic Morphologic Subtypes</td>
<td style="text-align: left;">Text term that describes various morphological and cytological subtypes in protate tumors.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sarcoma Subtype</td>
<td style="text-align: left;">The subtype related to the scientific determination and investigation, analysis and recognition of the presence and nature of disease, condition, or injury from expressed signs and symptoms of tissue growth resulting from uncontrolled cell proliferation.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sarcoma Diagnosis Classification Category</td>
<td style="text-align: left;">High level grouping to describe a diagnostic grouping or category for sarcoma, a malignant mesenchymal cell tumor most commonly arising from muscle, fat, fibrous tissue, bone, cartilage, and blood vessels.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Sarcoma Tumor Extension Type</td>
<td style="text-align: left;">The field to indicate the organs and structures to which the tumor has become adherent or has invaded.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pancreas Precancer Histopathologic Grade</td>
<td style="text-align: left;">The grade of precancerous pancreatic tissue based on microscopic study of characteristic tissue abnormalities by employing various cytochemical and immunocytochemical stains.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pancreatic IPMN Pathology Epithelial Subtype</td>
<td style="text-align: left;">The Intraductal Papillary Mucinous Neoplasm (IPMN) epithelial cell subtype based on the gross and microscopic examination of a pancreatic neoplasm specimen</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pancreatic Duct Final Pathology Type</td>
<td style="text-align: left;">The final pathology result of the pancreatic duct communication type.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cutaneous Melanoma Tumor Infiltrating Lymphocytes</td>
<td style="text-align: left;">Description of degree of lymphocytic infiltration surrounding and disrupting tumor cells of the vertical growth phase in a cutaneous melanoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cutaneous Melanoma Tumor Regression Range</td>
<td style="text-align: left;">Description of the degree to which tumor cells are replaced by lymphocytic inflammation with or without dermal melanophages and fibrosis._Range; the difference between the lowest and highest numerical values.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Melanoma Specimen Clark Level Value</td>
<td style="text-align: left;">Definition of the Clark level or depth of involvement of a melanoma in the skin or a specimen.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cutaneous Melanoma Surgical Margins</td>
<td style="text-align: left;">Text term to indicate presence of tumor at resection margins</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Melanoma Lesion Size</td>
<td style="text-align: left;">Diameter of lesion determined on skin examination (pre-bx), in mm</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">History of Atypical Nevi</td>
<td style="text-align: left;">Patient has a history of atypical nevi</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Fitzpatrick Skin Tone</td>
<td style="text-align: left;">The Fitzpatrick classification of skin phototype</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">History of Chronic UV Exposure</td>
<td style="text-align: left;">History of chronic UV exposure</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">History of Blistering Sunburn</td>
<td style="text-align: left;">Patient has history of blistering sunburn</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">History of Tanning Bed Use</td>
<td style="text-align: left;">History of tanning bed use of the patient</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Immediate Family History Melanoma</td>
<td style="text-align: left;">Text that describes the age at which the family member was diagnosed with melanoma skin cancer in relationship to their 50th birthday.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Melanoma Biopsy Resection Sites</td>
<td style="text-align: left;">Biopsy resection sites specific to melanoma (not covered in Tiers 1 and 2)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cutaneous Melanoma Ulceration</td>
<td style="text-align: left;">Description of extent of disruption to the surface of the skin caused by the cutaneous melanoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Cutaneous Melanoma Additional Findings</td>
<td style="text-align: left;">Significant pathologic finding present in addition to the cutaneous melanoma.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HTAN RPPA Antibody Table ID</td>
<td style="text-align: left;">HTAN identifier associated with RPPA antibody level metadata. Identical for every row of the table.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Ab Name Reported on Dataset</td>
<td style="text-align: left;">The antibody name.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">GENCODE Gene Symbol Target</td>
<td style="text-align: left;">The comma separated list of gene symbols targeted by the antibody.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">UNIPROT Protein ID Target</td>
<td style="text-align: left;">The comma separated list of UNIPROT IDs targeted by the antibody.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Phosphoprotein Flag</td>
<td style="text-align: left;">A flag the denotes if an antibody targets a phosphoprotein.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Internal Ab ID</td>
<td style="text-align: left;">Internal lab ID for an antibody.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Species</td>
<td style="text-align: left;">Host animal.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">RPPA Dilution</td>
<td style="text-align: left;">The dilution ratio.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Phospho Site</td>
<td style="text-align: left;">The protein site for a phosphoprotein targeting antibody. Report AA and site (i.e. S442)</td>
<td style="text-align: left;">Phosphoprotein Flag</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">RPPA Validation Status</td>
<td style="text-align: left;">Valid = RPPA and WB correlation &gt; 0.7; Use with Caution = RPPA and WB correlation &lt; 0.7; Under Evaluation = Antibody has given mixed results and/or evaluated by another lab; We are in the process of (re)validating; Used for QC = These antibodies are used for tissue sample quality control (QC)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Antibody Notes</td>
<td style="text-align: left;">Notes on antibodies replacements and antibody recognition observations.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pre-processing Completed</td>
<td style="text-align: left;">Pre-processing steps completed to convert level 1 raw data to a single level 2 image</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Pre-processing Required</td>
<td style="text-align: left;">Pre-processing steps required to convert level 1 raw data to a single level 2 image</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Publication</td>
<td style="text-align: left;">An empty parent attribute for publications</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Publication Manifest</td>
<td style="text-align: left;">Publication specific attributes.</td>
<td style="text-align: left;">Component,Publication-associated HTAN Parent Data File ID, HTAN Grant ID, HTAN Center ID, Publication Content Type, DOI, Title, Authors, Corresponding Author, Corresponding Author ORCID, Year of Publication, Location of Publication, Publication Abstract, License, PMID, Publication contains HTAN ID, Data Type, Tool, Supporting Link, Supporting Link Description</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Publication-associated HTAN Parent Data File ID</td>
<td style="text-align: left;">HTAN Data File Identifier(s) of the files associated with the content presented/published. Should be comma-separated lists.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Grant ID</td>
<td style="text-align: left;">HTAN grant number(s) (i.e. CA------ format) associated with the content presented/published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Center ID</td>
<td style="text-align: left;">List of HTAN Center ID(s) associated with the content presented/published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Publication Content Type</td>
<td style="text-align: left;">The type of content presented or published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DOI</td>
<td style="text-align: left;">The digital object identifier (DOI) of the content in the form of https://www.doi.org/{doi} to comply with CrossRef DOI display guidelines.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Corresponding Author</td>
<td style="text-align: left;">The name(s) of the corresponding author(s) of the content presented/published. If more than one corresponding author, please list in the order they appear in the author list.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Corresponding Author ORCID</td>
<td style="text-align: left;">The ORCiD(s) of the corresponding author(s) of the content presented/published. Should be a valid ORCiD url starting with https://orcid.org/ followed by a 16 digit identifier in dash separated groups of 4 (for example https://orcid.org/0000-0002-1825-0097). If more than one corresponding author, please list ORCiDs in the order the authors appear in the author list.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Title</td>
<td style="text-align: left;">The title of the content presented or published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Authors</td>
<td style="text-align: left;">The names of the author(s) of the content presented/published, in the order they appear.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Year of Publication</td>
<td style="text-align: left;">The year the content was presented or published (format YYYY).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Location of Publication</td>
<td style="text-align: left;">The name of the preprint server, journal, or conference where the content was presented/published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Publication Abstract</td>
<td style="text-align: left;">The abstract or short description of the content presented/published.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">License</td>
<td style="text-align: left;">The type of license applicable to the content.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">PMID</td>
<td style="text-align: left;">The PubMed identifier associated with the publication (applicable to published manuscripts). Provide as a URL of the form https://pubmed.ncbi.nlm.nih.gov/{pmid}</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Data Type</td>
<td style="text-align: left;">Types of data associated with the content. Fill out Other Data Type Specified, if not on the list.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Other Data Type Specified</td>
<td style="text-align: left;">Other types of data associated with the content.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Supporting Link</td>
<td style="text-align: left;">Relevant external links associated with the content (e.g external datasets used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and/or any other applicable agreement(s). Validated as URL</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Supporting Link Description</td>
<td style="text-align: left;">Description of relevant external links associated with the publication (e.g An external mouse dataset used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and-or any other applicable agreement(s).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tool</td>
<td style="text-align: left;">Were any software or computational tools generated for this content</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Accessory Data Type</td>
<td style="text-align: left;">Accesory specific data type</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Accessory</td>
<td style="text-align: left;">An empty parent attribute for accessory</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Accessory Manifest</td>
<td style="text-align: left;">Accessory specific attributes</td>
<td style="text-align: left;">Component,Dataset Name,Accessory Synapse ID,Accessory Description, Accessory Data Type,HTAN Center ID,HTAN Parent Biospecimen ID,Accessory-associated HTAN Parent Data File ID</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Dataset Name</td>
<td style="text-align: left;">Name of a dataset (e.g. a Synapse folder)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Accessory Synapse ID</td>
<td style="text-align: left;">Synapse ID of folder containing accessory files</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Accessory Description</td>
<td style="text-align: left;">Free text field containing description of accessory file(s)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Accessory-associated HTAN Parent Data File ID</td>
<td style="text-align: left;">HTAN Data File Identifier(s) of the files associated with the accessory content. Should be comma-separated lists.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MapQ30</td>
<td style="text-align: left;">Number of reads with Quality &gt;= 30.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">scATAC-seq Object ID</td>
<td style="text-align: left;">Orig.Ident or scATAC-seq Object ID</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">nCount Peaks</td>
<td style="text-align: left;">Total number of fragments in peaks</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">nFeature Peaks</td>
<td style="text-align: left;">Number of peaks with at least one read</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total Read-Pairs</td>
<td style="text-align: left;">Total read-pairs</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Duplicate Read-Pairs</td>
<td style="text-align: left;">Number of duplicate read-pairs</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Chimeric Read-Pairs</td>
<td style="text-align: left;">Number of chimerically mapped read-pairs</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Unmapped Read-Pairs</td>
<td style="text-align: left;">Number of read-pairs with at least one end not mapped</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">LowMapQ</td>
<td style="text-align: left;">Number of read-pairs with &lt;30 mapq on at least one end</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Mitochondrial Read-Pairs</td>
<td style="text-align: left;">Number of read-pairs mapping to mitochondria and non-nuclear contigs</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Passed Filters</td>
<td style="text-align: left;">Number of non-duplicate, usable read-pairs i.e. fragments</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">TSS Fragments</td>
<td style="text-align: left;">Number of fragments overlapping with TSS regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">DNase Sensitive Region Fragments</td>
<td style="text-align: left;">Number of fragments overlapping with DNase sensitive regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Enhancer Region Fragments</td>
<td style="text-align: left;">Number of fragments overlapping enhancer regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Promoter Region Fragments</td>
<td style="text-align: left;">Number of fragments overlapping promoter regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">On Target Fragments</td>
<td style="text-align: left;">Number of fragments overlapping any of TSS, enhancer, promoter and DNase hypersensitivity sites (counted with multiplicity)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Blacklist Region Fragments</td>
<td style="text-align: left;">Number of fragments overlapping blacklisted regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Peak Region Fragments</td>
<td style="text-align: left;">Number of fragments overlapping peaks</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Peak Region Cutsites</td>
<td style="text-align: left;">Number of ends of fragments in peak regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Nucleosome Signal</td>
<td style="text-align: left;">Nucleosome signal score (strength of the nucleosome signal per cell, computed as the ratio of fragments between 147 bp and 294 bp (mononucleosome) to fragments &lt; 147 bp (nucleosome-free))</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Nucleosome Percentile</td>
<td style="text-align: left;">Percentile rank of nucleosome score</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">TSS Enrichment</td>
<td style="text-align: left;">Transcription start site (TSS) enrichment score</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">TSS Percentile</td>
<td style="text-align: left;">Percentile rank of TSS score</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pct Reads in Peaks</td>
<td style="text-align: left;">Percentage of reads in peaks</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Blacklist Ratio</td>
<td style="text-align: left;">Ratio of reads in blacklist regions</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Seurat Clusters</td>
<td style="text-align: left;">Clusters of cells by a shared nearest neighbor (SNN) modularity optimization based clustering algorithm</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">nCount RNA</td>
<td style="text-align: left;">Total number of fragments in genes</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">nFeature RNA</td>
<td style="text-align: left;">Number of genes detected in cell</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Seqnames</td>
<td style="text-align: left;">Chromosome id</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Start</td>
<td style="text-align: left;">Genomic starting position in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 End</td>
<td style="text-align: left;">Genomic ending position in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Width</td>
<td style="text-align: left;">Width of the peak in bases in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Strand</td>
<td style="text-align: left;">DNA stand aligned with in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Name</td>
<td style="text-align: left;">Name of the peak in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Score</td>
<td style="text-align: left;">Peak score (proportional to q-value) in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Fold Change</td>
<td style="text-align: left;">Fold enrichment for this peak summit against random Poisson distribution with local lambda in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Neg Log10 pvalue Summit</td>
<td style="text-align: left;">Negative log10 p-value for the peak summit in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Neg Log10 qvalue Summit</td>
<td style="text-align: left;">Negative log10 q-value for the peak summit in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MACS2 Relative Summit Position</td>
<td style="text-align: left;">Position of the peak summit related to the start position in MACS2</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Is lowest level</td>
<td style="text-align: left;">Denotes that the manifest represents the lowest data level submitted. Use when L1 data is missing</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Is lowest level</td>
<td style="text-align: left;">If manifest is lowest level require HTAN Parent Biospecimen ID</td>
<td style="text-align: left;">HTAN Parent Biospecimen ID</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Normalization Method</td>
<td style="text-align: left;">Description of Normalization Process</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Batch Correction Method</td>
<td style="text-align: left;">Method that was used to batch correct Level 3 data</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">MS Batch ID</td>
<td style="text-align: left;">Batch ID indicating a set of samples that were run together.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS-based Assay Type</td>
<td style="text-align: left;">Analytes are the target molecules being measured with the assay.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS-based Targeted</td>
<td style="text-align: left;">Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay. Example: The MALDI Imaging analyte is lipids.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Instrument Vendor and Model</td>
<td style="text-align: left;">An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Source</td>
<td style="text-align: left;">The ion source type used for surface sampling (MALDI, MALDI-2, DESI, or SIMS) or LC-MS/MS data acquisition (nESI)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Polarity</td>
<td style="text-align: left;">The polarity of the mass analysis (positive or negative ion modes)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Mass Range Low Value</td>
<td style="text-align: left;">The low value of the scanned mass range for MS1 in m/z.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Mass Range High Value</td>
<td style="text-align: left;">The high value of the scanned mass range for MS1 in m/z.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Data Collection Mode</td>
<td style="text-align: left;">Mode of data collection in tandem MS assays. Either DDA (Data-dependent acquisition) or DIA (Data-indemendent acquisition.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Scan Mode</td>
<td style="text-align: left;">Indicates whether experiment is MS, MS/MS, or other (possibly MS3 for TMT)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Labeling</td>
<td style="text-align: left;">Indicates whether samples were labeled prior to MS analysis (e.g., TMT)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Instrument Vendor and Model</td>
<td style="text-align: left;">The manufacturer of the instrument used for LC.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Column Vendor and Model</td>
<td style="text-align: left;">The manufacturer of the LC Column unless self-packed, pulled tip capilary is used and the model number/name of the LC Column - IF custom self-packed, pulled tip calillary is used enter 'Pulled tip capilary'</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Resin</td>
<td style="text-align: left;">Details of the resin used for lc, including vendor, particle size, pore size</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Length Value</td>
<td style="text-align: left;">LC column length in cm.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Temp Value</td>
<td style="text-align: left;">LC temperature in C.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC ID Value</td>
<td style="text-align: left;">LC column inner diameter in microns.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Flow Rate</td>
<td style="text-align: left;">LC flow rate in nL/min.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Gradient</td>
<td style="text-align: left;">The program dictates the mobile phase solvent composition over the course of the chromatographic run.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Mobile Phase A</td>
<td style="text-align: left;">Composition of mobile phase A</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">LC Mobile Phase B</td>
<td style="text-align: left;">Composition of mobile phase B</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Instrument Metadata File</td>
<td style="text-align: left;">Additional file containing instrument metadata details. Use either synapse_path or entity_Id</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Bisulfite Conversion</td>
<td style="text-align: left;">Name of the kit used in bisulfite conversion.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Replicate Type</td>
<td style="text-align: left;">A common term for all files belonging to the same sample. We suggest using a stable sample accession from a biosample archive like BioSamples.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Bulk Methylation Assay Type</td>
<td style="text-align: left;">Assay types normally determine genomic coverage</td>
<td style="text-align: left;">Targeted Genome, Beadchip Array</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Targeted Genome</td>
<td style="text-align: left;">Assay for analyzing specific mutations in a given sample</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Beadchip Array</td>
<td style="text-align: left;">Assay that uses beads to target a specific locus on the genome.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Total DNA Input</td>
<td style="text-align: left;">Overall number of reads for a given sample in digits (microgram, nanogram).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Trimmer</td>
<td style="text-align: left;">Software used for trimming</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Bulk Methylation Genomic Reference</td>
<td style="text-align: left;">The human genome reference used in the alignment of reads</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Duplicate Removal Software</td>
<td style="text-align: left;">Software used for remove duplicate reads</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Proportion of Minimum CpG Coverage 10X</td>
<td style="text-align: left;">Proportion of all reference bases for whole genome sequencing, or targeted sequencing, that achieves 10X or greater coverage per CpG.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">DMC Calling Tool</td>
<td style="text-align: left;">Software used for calling differentially methylated CpG (DMC) and differentially methylated region (DMR)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DMC Calling Workflow URL</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DMR Calling Tool</td>
<td style="text-align: left;">Software used for calling differentially methylated CpG (DMC) and differentially methylated region (DMR)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DMR Calling Workflow URL</td>
<td style="text-align: left;">Generic name for the workflow used to analyze a data set</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">pUC19 methylation ratio</td>
<td style="text-align: left;">Methylation ratio of mostly methylated pUC19 control, as a percentage</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Lambda methylation ratio</td>
<td style="text-align: left;">Methylation ratio of mostly unmethylated lambda control, as a percentage</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DMC data file format</td>
<td style="text-align: left;">Format of the data files</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">DMR data file Format</td>
<td style="text-align: left;">Format of the data files.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">MS Assay Category</td>
<td style="text-align: left;">Type of Mass Spectrometry performed.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Publication contains HTAN ID</td>
<td style="text-align: left;">HTAN IDs are used in the publication.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Electron Microscopy Level 1</td>
<td style="text-align: left;">Raw electron microscopy data as one TIFF file per plane for a 3D image stack or per tile for a 2D large area montage</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, EM method, EM signal or contrast mech, EM instrument, Protocol Link, Software and Version, SizeX, SizeY, SizeC, SizeZ, PhysicalSizeX, PhysicalSizeY, PhysicalSizeZ, EM dwell or exposure time,EM voltage, EM beam current, EM spot size, EM stage tilt, EM signal processing, EM contrast type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Electron Microscopy Level 2</td>
<td style="text-align: left;">Processed electron microscopy data as one OME-TIFF image per plane or montage</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID,Tile overlap X, Tile overlap Y,EM contrast type</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Electron Microscopy Level 3</td>
<td style="text-align: left;">Segmented electron microscopy data as .am or .tiff formats</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Imaging Object Class</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Electron Microscopy Level 4</td>
<td style="text-align: left;">Movies or other derived files from electron microscopy data</td>
<td style="text-align: left;">Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Comment</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM instrument</td>
<td style="text-align: left;">Make and model of the EM instrument used</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">EM method</td>
<td style="text-align: left;">Electron microscopy method used</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">EM signal or contrast mech</td>
<td style="text-align: left;">How the electron microscopy signal is generated from the sample</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">EM dwell or exposure time</td>
<td style="text-align: left;">Duration in microseconds (µs) of electron beam data collection per pixel or frame</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM voltage</td>
<td style="text-align: left;">Accelerating voltage in kiloelectronvolts (keV)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM beam current</td>
<td style="text-align: left;">Beam current in nanoamps (nA)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM spot size</td>
<td style="text-align: left;">Beam spot size in micrometers (µm)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM stage tilt</td>
<td style="text-align: left;">Physical stage tilt in degrees with respect to the electron beam</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM signal processing</td>
<td style="text-align: left;">SNR improvement strategies used</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">EM contrast type</td>
<td style="text-align: left;">Does the image use standard SEM contrast or TEM contrast</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Tile overlap X</td>
<td style="text-align: left;">Percentage of image overlap to allow tile stitching in x direction</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Tile overlap Y</td>
<td style="text-align: left;">Percentage of image overlap to allow tile stitching in x direction</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Barretts Esophagus Goblet Cells Present</td>
<td style="text-align: left;">Presence or absennce of Barretts esophagus goblet cells.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Pancreatitis Onset Year</td>
<td style="text-align: left;">Date of onset of pancreatitis.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">HTAN Parent Channel Metadata ID</td>
<td style="text-align: left;">HTAN ID for a level 3 channels table.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Single Nucleus Capture</td>
<td style="text-align: left;">Nuclei isolation method</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Associated mRNA Library Data File ID</td>
<td style="text-align: left;">Sample Level HTAN Data File ID for the associated level - HTAN ID of this file HTAN ID SOP (eg HTANx_yyy_zzz)</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Single Cell Barcode Method Applied</td>
<td style="text-align: left;">The method by which cells are multiplex or labeled with cell surface markers or probes</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Feature Barcode Library Type</td>
<td style="text-align: left;">The library construction methods for the feature barcode library</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Barcode Folder Synapse ID</td>
<td style="text-align: left;">Synapse ID of the folder containing the barcode lists</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Barcode Folder File List</td>
<td style="text-align: left;">A comma separated list of filenames in the gzipped folder detailing what barcodes are specific to demultiplexing samples versus providing surface protein data</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Platform ID</td>
<td style="text-align: left;">The NCBI GEO Microarray Platform ID that links to the table containing the array definition</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Molecule</td>
<td style="text-align: left;">Microarray is measuring this kind of molecule</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Label</td>
<td style="text-align: left;">Microarray used this kind of label</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Value Definition</td>
<td style="text-align: left;">What the provided value signifies</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Microarray Protocol Auxiliary File</td>
<td style="text-align: left;">Auxiliary file describing the experimental protocols used, as described in the NCBI GEO microarray template, recorded as synapse ID (syn12345).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Participant Vital Status Update</td>
<td style="text-align: left;">Updates to a participants vital status</td>
<td style="text-align: left;">Component, HTAN Participant ID, Vital Status</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Precancer Diagnosis</td>
<td style="text-align: left;">Diagnosis of a precancerous condition</td>
<td style="text-align: left;">Component, HTAN Participant ID, Precancer Case</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Alive</td>
<td style="text-align: left;">This indicates the participant is alive and defines further required metadata</td>
<td style="text-align: left;">Days to Vital Status Reference</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Vital Status Reference</td>
<td style="text-align: left;">Number of days between the date used for index and the reference date for designation of vital status</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Precancer Case</td>
<td style="text-align: left;">Yes/No indicator to designate the participant for whom precancerous lesion(s) was identified (premalignancy only).</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">True</td>
</tr>
<tr>
<td style="text-align: left;">Yes - Precancer Case</td>
<td style="text-align: left;">Indicates that the participant is a precancer case</td>
<td style="text-align: left;">Precancerous Condition Type, Days to Precancer Case Designation, WHO Precursor Lesion Code</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">Days to Precancer Case Designation</td>
<td style="text-align: left;">Number of days between the date used for index and the reference date for designation of precancer status.</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
<tr>
<td style="text-align: left;">WHO Precursor Lesion Code</td>
<td style="text-align: left;">World Health Organization Classification of Tumour cytopathology-based coding system, includes 'precursor lesion' designations for precancers. ICD-O-3 morphology axis format eg 1234/1</td>
<td style="text-align: left;">nan</td>
<td style="text-align: left;">False</td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
      
      <nav class="md-footer__inner md-grid" aria-label="Footer" >
        
          
          <a href=".." class="md-footer__link md-footer__link--prev" aria-label="Previous: Home">
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
            </div>
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Previous
              </span>
              <div class="md-ellipsis">
                Home
              </div>
            </div>
          </a>
        
        
      </nav>
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["content.code.copy", "navigation.tabs", "navigation.footer", "navigation.sections", "search.highlight", "search.share", "search.suggest", "toc.follow"], "search": "../assets/javascripts/workers/search.6ce7567c.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.88dd0f4e.min.js"></script>
      
    
  </body>
</html>